\documentclass[a4paper,10pt]{article}

\usepackage[margin=0.8in]{geometry}
\usepackage[T1]{fontenc}
\usepackage[english]{babel}

\usepackage{color}
\definecolor{linkColor}{rgb}{0.2,0.4,0.6}

\usepackage[
	colorlinks,
	citecolor=linkColor,
	linkcolor=linkColor,
	menucolor=linkColor,
	urlcolor=linkColor,
]{hyperref}

\usepackage{amsfonts}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{array}
\usepackage{bold-extra}
\usepackage{enumitem}
\usepackage{longtable}
\usepackage{url}
\usepackage{xcolor}
\usepackage{eurosym}
\usepackage{lmodern} 

%%%% TITLE SETTING
\newcommand{\bigtitle}[1]{
	{\begin{center} 
		\huge \textsc{#1}\\[0.7ex]
		\hrule
	\end{center}}
}
\newcommand{\mediumtitle}[1]{
	\vspace{0.2cm}
	{\noindent
	\Large \textsc{#1}\\[-2ex]
	\hrule
	\vspace{0.2cm}}
}
\newcommand{\smalltitle}[1]{
	\vspace{0.1cm}
	{\noindent 
	\large \textsc{#1}}
	\vspace{0.1cm}
}


%%%% PARAGRAPH
\newcommand{\indentp}[1]{
	\vspace{0.2cm}
	\begin{minipage}{0.95\textwidth}
	{\noindent \hskip-1px #1}
	\end{minipage}
	\vspace{0.2cm}
}

%%%% LIST
%double item
\newcolumntype{A}{>{\raggedleft}p{0.22\textwidth}}
\newcolumntype{B}{p{0.7\textwidth}}
\newenvironment{doubletablelist}
{
	\vspace{-0.2cm}
	\begin{longtable}[!h]{AB}}{\end{longtable}
}
\newcommand{\dtlist}[2]{
\hspace{-3cm}
\noindent
	\begin{minipage}{0.22\textwidth}
	\begin{flushright}
	\textsc{#1}
	\end{flushright}
	\end{minipage}
	& #2\\[0.2cm]
}
%single item
\newcolumntype{a}{>{\raggedleft}p{0.14\textwidth}}
\newcolumntype{b}{p{0.8\textwidth}}
\newenvironment{singletablelist}
{	\vspace{-0.2cm}
	\begin{longtable}[!h]{ab}}{\end{longtable}
}
\newcommand{\stlist}[2]{
	\hspace{-3cm}
	\noindent
	\begin{minipage}{0.24\textwidth}
	\begin{flushright}
	\textsc{#1}
	\end{flushright}
	\end{minipage}
	& #2\\[0.2cm]
}

%%%% ITEM BULLET
\newcommand{\bulletitem}{\item[$\bullet$]}
\newcommand{\graybulletitem}{ \item[\textcolor{gray}{$\bullet$}]}
\newcommand{\circitem }{\item[$\circ$]}
\newcommand{\minusitem}{\item[-]}
\newcommand{\astitem}{\item[$\ast$]}
\newcommand{\noitem}{\item[]}

% --------------------------------

\begin{document}

\begin{center}
\bigtitle{Alessia Visconti, PhD}
\end{center}

\vspace{0.2cm}

\noindent
\textbf{Research Fellow}, Department of Twin Research \& Genetics Epidemiology\\
King's College London, St Thomas' Hospital Campus, 3rd Floor South Wing Block D\\
Westminster Bridge Road, London SE1 7EH

\noindent
\textsc{e-mail:} \url{alessia.visconti@kcl.ac.uk} \\

%\vspace{0.2cm}

\mediumtitle{Research Interests}
\begin{itemize}[itemsep=-0.5ex]
 	\minusitem  \textsc{Computational Biology \& Medicine}
 	\minusitem  \textsc{Data Mining \& Machine Learning}
	% \minusitem  \textsc{High-Throughput Biology}
	% \minusitem  \textsc{Complex Disease Genetics}
	% \minusitem  \textsc{Big Data}
	\minusitem  \textsc{Research Software Engineering}
\end{itemize}

\mediumtitle{Brief Synopsis of Research}
\indentp{
\noindent Alessia Visconti is an expert in bioinformatics and genetic epidemiology, and her research activity deals with the development and application of statistical and computational methods to identify multi\emph{-omics} modifications influencing complex human phenotypes.
She has also worked on the problem of knowledge discovery in biological data, developing new approaches tailored to solve biological tasks, and on the reverse engineering of gene regulatory networks. 
}

\vspace{0.4cm}

% --------------------------------
% EDUCATION
% --------------------------------

\mediumtitle{Education}
\begin{singletablelist}
	%\stlist{Jul 2012}{\textbf{PhD in Science and High Technology}, University of Turin.\\
	\stlist{Jul 2012}{\textbf{PhD in Computer Science}, University of Turin.\\
	&\textsc{Thesis title:} \emph{Systems Biology: Knowledge Discovery and Reverse Engineering} }
	% &\textsc{Advisors}: prof. Marco Botta, Dr. Roberto Esposito}
	
	\stlist{Jul 2008}{\textbf{Master degree in Computer Science} \emph{``summa cum laude''}, University of Turin.\\
	&\textsc{Thesis title:} \emph{SPOT: an algorithm for the extraction and the analysis of biological patterns} }
	% &\textsc{Advisor}: prof. Marco Botta}

	\stlist{Mar 2006}{\textbf{Bachelor degree in Computer Science}  \emph{``summa cum laude''}, University of Turin.\\
	&\textsc{Thesis title:} \emph{The Haskell language} }
	% &\textsc{Advisor}:  prof. Viviana Bono}
\end{singletablelist}

% --------------------------------
% SKILLS
% --------------------------------

\mediumtitle{Skills}
\begin{doubletablelist}
	\dtlist{Language Skills}{\textsc{Italian}: native proficiency \\ 
							&\textsc{English}: professional working proficiency\\
							&\textsc{Greek (modern)}: elementary proficiency}
	\dtlist{Computing Skills}{\textsc{Programming languages}: bash, C, C++, JAVA, php, python, R, ruby\\
							&\textsc{Other languages}: CSS, \LaTeX, HTML, PyQt, XML\\
							&\textsc{Statistical software}: R, SAS\\
							&\textsc{Database management}: MySQL, MariaDB\\
							&\textsc{Version control systems \& reproducibility }: GIT, nextflow, docker, singularity\\
							&\textsc{Bioinformatics \& Genetic analysis}: BBmap, BEDTools, DESeq2, GCTA, GWAMA, LDAK, limma, lmekin, metal, Merlin, PLINK, QTDT, samtools, vcftools, \dots\\
							&\textsc{Structural Equation Modelling}: openMX, Mplus\\
							&\textsc{Data visualisation}: dot, ggplot2}
\end{doubletablelist}

\newpage

% --------------------------------
% RESEARCH
% --------------------------------

\mediumtitle{Research posts}
\begin{doubletablelist}
    \dtlist{Aug 2017 - Present}{\textbf{Research fellow} at the Department of Twin Research \& Genetic Epidemiology, King's College London, UK \\	
	 &\textsc{Research activity}: In her present role, Alessia Visconti is responsible for (a) the development and application of statistical and computational multi\emph{-omics} approaches to understand the mechanisms and to improve the diagnosis and treatment of human diseases, (b) the supervision of master and PhD students as well as postdocs, (c) the writing research grants, and (d) the scientific responsibility of research projects.\\
	  & From her previous role, Alessia Visconti continues to led the bioinformatics analyses for a set of studies aiming at dissecting the aetiology of melanoma and its risk phenotypes~\cite{Vis19a,Vis20,Lan20,San20} and at studying IgA Nephropathy~\cite{Dot21}. She is also responsible for several new projects aiming at investigating the influence of the gut microbiome on human health~\cite{Vis19,Bar20,LeR22,Zha22,Val23,Lou23}, predicting melanoma response to immunotherapy~\cite{Ros22,Vis23}, and at studying thyroid diseases~\cite{Mar20}, atopic dermatitis~\cite{Gro21,Bud22}, cardiovascular diseases and their risk factors~\cite{Ros21}, immune system modifications~\cite{Pia21}, and X-inactivation~\cite{Zit23}. During the SARS-CoV-2 pandemic, she collaborated with the ZOE COVID Study (now ZOE Health Study), to (a) develop the analysis pipeline for the daily data provided by more than four million users~\cite{Mur21}, (b) led two studies investigating skin manifestations of SARS-CoV-2~\cite{Vis21, Vis22}, and (c) performed the bioinformatics analysis for several other studies~\cite{Men20,Lee20,Zaz20,Hop21,Wil21,Sud21}. %\\
	% &\textsc{Supervisor}: Dr. Mario Falchi
	}
    \dtlist{Jun 2016 - Jun 2019}{\textbf{Honorary research associate} at CERN, Switzerland \\
	 &\textsc{Research activity}: During her honorary post at the CERN Openlab, Alessia Visconti extended the ROOT library %, developed by the CERN OpenLab for enabling the storage and scientific analyses and visualisation of large amounts of data from particle physics experiments, 
	 to allow the efficient storage of genomic data.  %\\
	% &\textsc{Supervisor}: Dr. Marco Manca
	}
    \dtlist{Apr 2015 - Jul 2017}{\textbf{Research associate} at the Department of Twin Research \& Genetic Epidemiology, King's College London, UK \\ 
	&\textsc{Research activity}: During this post, Alessia Visconti was mostly responsible for the development and application of statistical and computational multi\emph{-omics} approaches to understand the mechanisms and to improve the diagnosis and treatment of human diseases. She also supervised master and PhD students and contributed to the writing of research grants.\\
	 & In this role, Alessia Visconti led the bioinformatics analyses for a set of projects aiming at dissecting the aetiology of melanoma, melanoma risk phenotypes, and their connection with ageing~\cite{Rib16,Hys18,Vis18a,Duf17}, and at studying IgA Nephropathy~\cite{Lom16}, cognition and neurodevelopmental disease~\cite{Cul18}, and epigenetic plasticity~\cite{Car16} and modification~\cite{Zag18}. She also reported on how to conduct metagenomic studies in microbiology and clinical research~\cite{Vis18c} and developed a novel pipeline which ensures reproducibility in metagenomics research~\cite{Vis18b}.%\\
	% &\textsc{Supervisor}: Dr. Mario Falchi
	}
    \dtlist{Jan 2014 - Mar 2015}{\textbf{Research associate} at the Department of Genomics of Common Disease, School of Public Health, Imperial College London, UK  \\
	 &\textsc{Research activity}: During this post, Alessia Visconti (a) developed and implemented a novel approach for the population and pedigree association testing for quantitative data~\cite{Vis16}, and (b) conducted the bioinformatics analysis for~\cite{Joh15,AlM15,Gia16,Pui16}.%\\
	% &\textsc{Supervisor}: Dr. Mario Falchi
	}
	\dtlist{Jan 2012 - Dec 2013}{\textbf{Research associate} at the Department of Computer Science, University of Turin, Italy \\ %Recipient of a 2-year fellowship for her research proposal, awarded by the Italian Minister of Education, University and Research. \\
	 &\textsc{Research activity}: During this post, Alessia Visconti: (a) developed and implemented a novel bi-clustering approach leveraging additional knowledge~\cite{Vis13a}, (b) contributed to the development of a novel exact algorithm for answering Maximum a Posteriori queries on tree structures~\cite{Esp13}, (c) applied machine learning approaches for the prediction and interpretation of the lipophilicity of small peptides~\cite{Vis15a}, and (d) conducted the bioinformatics analysis to measure the ability of more than 200 compounds of acting as hydrogen bond donors~\cite{Erm14}.%\\
	% &\textsc{Supervisors}: Dr. Roberto Esposito
	}
	\dtlist{Jun 2011 - Dec 2011}{\textbf{Visiting researcher} at the Center of Biological Sequence Analysis, Technical University of Denmark, Denmark 
	\\ % at the Regulatory Genomics Group at the Center of Biological Sequence Analysis, Systems Biology Department, Technical University of Denmark, Denmark 
		  &\textsc{Research activity}: During her stay, Alessia Visconti (a) continued her PhD project developing a new method for the reverse engineering of gene regulatory networks that uses a popular econometrics statistical hypothesis test, namely the Granger Causality~\cite{Vis12b}, and (b) developed an \emph{ensemble} approach for the prediction of promoter activity. %\\
	% &\textsc{Supervisor}: Prof. Christopher Workman
	}
	\dtlist{Jan 2009 - Dec 2011}{\textbf{PhD student} at Department of Computer Science, University of Turin, Italy \\
	  &\textsc{Research activity}: During her PhD, Alessia Visconti developed and implemented: (a) a \emph{de novo} framework (accompanied by a web interface) performing protein motifs identification and allowing the simultaneous associations between groups of protein sequences and groups of motifs thanks to a constrained co-clustering approach~\cite{Cor09a}, (b) a new methodology (accompanied by a web interface) that provides meaningful co-clusters whose discovery and interpretation are enhanced by embedding gene ontology (GO) annotations~\cite{Vis11c,Cor09b}, (c) a novel algorithm for the rewriting of the GO aiming at obtaining a more compact and informative ontology~\cite{Vis11a,Vis10a}, (d) two algorithms for the reverse engineering of gene regulatory networks~\cite{Vis11b,Mar12,Vis12b}, and (c) contributed to a modular framework for the analysis of metagenomics sequences leading the co-clustering module development~\cite{Bon11}. %\\
	% &\textsc{Supervisors}: Dr. Roberto Esposito, Prof. Marco Botta
	}
	\dtlist{Sep 2008 - Dec 2008}{\textbf{Research assistant} at the Department of Computer Science, University of Turin and in collaboration with the Department of Arboriculture and Pomology, University of Turin, Italy  \\
	 &\textsc{Research activity}: During this post, Alessia Visconti contributed to the development of computational approaches for the classification and traceability of fruits produced in Piemonte. %\\
	% &\textsc{Supervisors}: Prof. Marco Botta, Prof. Roberto Botta
	}
\end{doubletablelist}

\mediumtitle{Career breaks}
\begin{doubletablelist}
    \dtlist{Aug 2022 - Apr 2023}{\textbf{Maternity leave}
	}
\end{doubletablelist}

\mediumtitle{Awarded fellowships}

\begin{doubletablelist}
    \dtlist{Jan 2012 - Dec 2013}{\textbf{Two years postdoctoral fellowship} awarded by the Italian Minister of Education, University and Research} 
	\dtlist{Jan 2012 - Dec 2012}{\textbf{Postdoctoral training grant} awarded by the Regione Piemonte} 
	\dtlist{Jan 2009 - Dec 2011}{\textbf{Three years PhD fellowship} awarded by the Italian Minister of Education, University and Research (best PhD project)}
\end{doubletablelist}



% --------------------------------
% PROJECTS
% --------------------------------

\mediumtitle{Research collaborations}

\begin{doubletablelist}
    \dtlist{2022-2025}{\begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Challenging the Dogma of Homogeneity in Gestational Diabetes Mellitus''}\\
		\textsc{Funder:} \emph{MRC Medical Research Council}\\
		\textsc{Role:} Named researcher, co-writer of the proposal (total grant funded: £1,090,268). In this project, which is now in the phase of data collection and aims at characterising pathophysiologically distinct subtypes of gestational diabetes (GDM), Alessia Visconti will develop and use bioinformatics and machine learning approaches to (a) evaluate similarities and differences in GDM subtypes in women of White European and South Asian descent, identifying variables (clinical/biochemical) that distinguish between subtypes, and (b) explore the relationships between subtypes and maternal/fetal/neonatal outcomes.
	\end{minipage}}
	 
	\dtlist{2022-2023}{\begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``A Collaborative Approach to the Borne Uterine Mapping Programme (BUMP) Feasibility Study''}\\
		\textsc{Funder:} \emph{BORNE}\\
		\textsc{Role:} Named researcher, co-writer of the proposal (total grant funded: £500,000). In this project, which is now in the first phase of data collection, Alessia Visconti will develop (a) pipelines for the analysis of single-cell and single-nucleus RNA sequencing and spatial transcriptomic data, and (b) deep-learning approaches for modelling their interaction. 
	\end{minipage}} 

	\dtlist{2021-2024}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Understanding phenotype and mechanisms of spontaneous preterm birth in sub-Saharan Africa (PRECISE-SPTB)''}\\
		\textsc{Funder:} \emph{MRC Medical Research Council}\\
		\textsc{Role:} Named bioinformatics instructor,  contribution to the project proposal (total grant funded: £458,800).  Within the project, which aims at determining the epidemiological and contextual nature of spontaneous preterm birth in three sub–Saharan African countries (Kenya, The Gambia and Mozambique), while developing technical infrastructure and training research scientists, Alessia Visconti is responsible for the preparation and delivery of a set of workshops which cover both basic and specialistic skills, namely: shell programming, version control and collaboration with Git/Github, programming in R, programming in python, workflow development with Nextflow, machine-learning approaches for biomedical data analysis, and metagenomic data analysis. Workshops delivered so far have been rated as “exceptional” by the attendees.
	\end{minipage}}
	
	\dtlist{2021-2022}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Predicting Response to Immunotherapy for Melanoma with gut Microbiome and metabolomics - The PRIMM Study''}\\
		\textsc{Funder:} \emph{Seerave Foundation}\\
		\textsc{Role:} Researcher. Within the project, which aims at finding novel biomarkers of response and survival to identify those patients with melanoma who are most likely to benefit from immune checkpoint inhibitors, Alessia Visconti led the bioinformatics analysis for the identification of glyco-markers~\cite{Vis23} and collaborated to the analysis of proteomic data~\cite{Ros22}.
	\end{minipage}}

	\dtlist{2020-2021}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``A multi-omics study to dissect the role of the gut microbiome in IgA nephropathy risk''}\\
		\textsc{Funder:} \emph{King's College London - Peking University Health Science Centre Joint Institute for Medical Research}\\
		\textsc{Role:} Named researcher, co-writer of the proposal (total grant funded: £74,000). Within this project, Alessia Visconti was performing \emph{in silico} characterisation and validation, using bioinformatics models, of microbes associated with IgA nephropathy and/or IgA glycosylation profiles.
	\end{minipage}}
	
	\dtlist{2019-2021}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Dissecting the mechanisms of immune-mediated inflammation: a systems-immunology approach''}\\
		\textsc{Funder:} \emph{MRC Medical Research Council}\\
		\textsc{Role:} Researcher. Within this project, Alessia Visconti performed bioinformatics analyses aiming at (a) the reverse engineering of immune cell co-expression networks and their involvement in a set of autoimmune diseases, and (b) the identification of genetic variations and of microbes and metabolites responsible for the development of such diseases.
	\end{minipage}}
	
	\dtlist{2016-2018}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Gut microbiome modulation of fasting glucose homeostasis and postprandial glycaemic response in TwinsUK and PREDICT: towards personalised diet for healthy aging''}\\
		\textsc{Funder:} \emph{Chronic Disease Research Foundation}\\
		\textsc{Role:} Named researcher. Within this project, Alessia Visconti (a) developed a tool for the analysis of metagenomic data which ensures the reproducibility of the scientific results~\cite{Vis18b}, (b) performed the bioinformatics analysis of metagenomics and metabolomics data~\cite{Vis19,Bar20}, (c) collaborated to five further studies (~\cite{Lou23}, or under revision), and (d) supervised PhD student Xinyuan Zhang for the work described in~\cite{Zha22}, of which she is co-senior author.
	\end{minipage}}
	
	\dtlist{2016-2018}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``A high-resolution map of copy number and structural variation in Qatari genomes and their contribution to quantitative traits and disease''}\\
		\textsc{Funder:} \emph{Qatar Foundation}\\
		\textsc{Role:} Named researcher, co-writer of the proposal (total grant funded: £160,521). Within this project, Alessia Visconti developed (a) an approach for the storage of genomic data taking advantage of the ROOT library, and (b) an ensemble approach for the identification of structural variation. She also conducted the bioinformatics analysis for~\cite{Ros21} and supervised PhD Niccol\`o Rossi.
	\end{minipage}}
	
	\dtlist{2014-2016}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``An integrative genomics approach for non-invasive diagnostic biomarkers discovery in IgA nephropathy''}\\
		\textsc{Funder:} \emph{MRC Medical Research Council}\\
		\textsc{Role:} Researcher. Within this project, Alessia Visconti applied statistical and bioinformatics approaches for studying the role of IgA and its glycosylation profiles in the development of IgA nephropathy, as described in~\cite{Lom16,Dot21}.
	\end{minipage}}
	
	\dtlist{2013-2015}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Senescence and melanoma -- An integrative systems biology approach to characterise the link between reduced biological senescence and melanoma susceptibility''}\\
		\textsc{Funder:} \emph{British Skin Foundation}\\
		\textsc{Role:} Researcher. Within this project, Alessia Visconti applied statistical and bioinformatics approaches for studying melanoma, melanoma risk phenotypes, and their connection with ageing, as described in~\cite{Rib16,Hys18,Vis18a,Duf17,Vis19a,Vis20,San20}.
	\end{minipage}}
		
	\dtlist{2013-2018}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Genomic analysis of Type 2 Diabetes in Qatar, towards diabetes personalized medicine''}\\
		\textsc{Funder:} \emph{Qatar Foundation}\\
		\textsc{Role:} Researcherr. Within this project, Alessia Visconti (a) developed and implemented an approach for the population and pedigree association testing for quantitative data~\cite{Vis16}, and (b) conducted the bioinformatics analysis for~\cite{AlM15} and~\cite{Zag18}.
	\end{minipage}}
	
	\dtlist{2012-2013}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``LIMPET -- Isotropic And Anisotropic Lipophilicity To Model Permeability Of New Therapeutic Peptides''}\\
		\textsc{Funder:} \emph{Compagnia di San Paolo}\\
		\textsc{Role:} Named researcher, co-writer of the proposal. Within this project, Alessia Visconti (a) evaluated the ability of some combinations of descriptors/algorithms to find the best model to predict the lipophilicity of small peptides~\cite{Vis15a}, and (b) performed the bioinformatics analyses to measure the ability of more than 200 compounds of acting as hydrogen bond donors~\cite{Erm14}.
	\end{minipage}}	

	\dtlist{2007-2011}{   \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``BioBITs -- Developing white and green biotechnologies by converging platforms from biology and information technology towards metagenomics''}\\
		\textsc{Funder:} \emph{Regione Piemonte}\\
		\textsc{Role:} Researcher Within this project, Alessia Visconti contributed to the development of a modular framework for the analysis of metagenomics sequences, and was responsible for the co-clustering module~\cite{Bon11}. 
	\end{minipage}}
	
	\dtlist{2004-2009}{  \begin{minipage}[t]{0.65\textwidth}
		\textsc{Title:} \emph{``Realizzazione di modelli informatici per la valorizzazione della qualit\`a e la tracciabilit\`a delle produzioni in specie da frutto coltivate in Piemonte''}\\
		\textsc{Funder:} \emph{Regione Piemonte}\\
		\textsc{Role:} Developer. Within this project, Alessia Visconti contributed to the development of computational approaches for the classification and traceability of fruits produced in Piemonte.
	\end{minipage}}
			
\end{doubletablelist}

\vspace{0.4cm}

% --------------------------------
% SOFTWARE
% --------------------------------

\newpage

\mediumtitle{Development of research software}

{\small \noindent The software is available at \url{https://github.com/alesssia} or upon request.}

\vspace{0.2cm}

\begin{doubletablelist}
	\vspace{-1cm}
	\dtlist{AID-ISA}{extracts biologically relevant biclusters from microarray gene expression data by leveraging additional knowledge (described in~\cite{Vis13a})}
	\dtlist{CDoT}{is a novel exact algorithm for answering Maximum a Posteriori queries on tree-structured Probabilistic Graphical Models (described in~\cite{Esp13})}
	\dtlist{famCNV (v2.0)}{enables genome-wide association of copy number variants with quantitative phenotypes in families}
	\dtlist{GOClust}{performs a co-clustering of microarray gene expression data using Gene Ontology-derived constraints (described in~\cite{Cor09b})}
	\dtlist{MotifsLinker}{associates clusters of proteins with their frequent motifs (described in~\cite{Cor08a})}
	\dtlist{PopPAnTe}{enables pairwise association testing in related samples (described in~\cite{Vis16})}
	\dtlist{RGO}{is a reorganization of the Gene Ontology emphasising  regulative information and providing better structure for gene functional analysis (described in~\cite{Vis11a})}
	\dtlist{SPOT}{performs an exhaustive search of frequent motifs in sets of biological sequences (described in~\cite{Vis08})}
	\dtlist{YAMP}{allows processing raw metagenomic sequencing data up to the functional annotation (described in~\cite{Vis18b})}
\end{doubletablelist}

\vspace{0.2cm}



% --------------------------------
% CHALLENGES

\mediumtitle{Participation in international challenges}

\begin{doubletablelist}
\vspace{-0.1cm}
\dtlist{July 2018}{\begin{minipage}[t]{0.65\textwidth}
			\emph{``BioDataHack 2018 -- Genomic, Biodata and Improving Health Outcomes''}\\
			%\textsc{Team:} Visconti A., Jun Aruga, Oliver Giles, Ioannis Valasakis, Chen Zhang\\
			\textsc{Rank:} The project presented by Alessia Visconti and the other members of her team (Jun Aruga, Oliver Giles, Ioannis Valasakis e Chen Zhang) advanced the vision of a device that will allow the constant monitoring of IBDs by patients from the comfort of their own homes, ranked first on the ARM, Cavium, and Atos Challenge: \emph{How can we use mobile technology to transform biological data processing?}. During the two-day BioData Hackathon, the team successfully ported the metagenomics pipeline developed by Alessia Visconti~\cite{Vis18b} onto Arm’s 64-bit architecture, where the data could be processed in a few hours, showing that the analysis of microbial data can be successfully taken out of centralised data centres. The solution also implemented a neural network that, receiving as input the microbial profile produced by the analysis pipeline, could predict the disease status. Even with a very small set of training data, the proposed approach was able to separate patients with Crohn's disease and ulcerative colitis (the two main forms of IBD) from healthy controls with over 90\% accuracy. The team also proposed a derivative score from the neural network’s output. Such a score, which could be tracked over time, would allow individuals to measure the effects of lifestyle/medication changes on their disease’s progression to assess their efficacy in almost real time. The next step would be to use the data generated in this monitoring process to craft suggestive models, able to offer individuals advice and treatments based on what has been effective in patients with similar microbiome profiles.
		\end{minipage}}

\dtlist{Summer 2011}{\begin{minipage}[t]{0.65\textwidth}
			\emph{``DREAM6 -- Promoter Activity Prediction Challenge''}\\
			%\textsc{Team:} Visconti A. (University of Turin) and Workman C.T. (Technical University of Denmark)\\
			\textsc{Rank:} The proposed approach (developed with Ali Altıntas and Chris Workman) which combined results from two well-known machine learning approaches (regression trees and support vector machines for regression), ranked 8th out of 21 participants.
		\end{minipage}}
		
\dtlist{Summer 2010}{\begin{minipage}[t]{0.65\textwidth}
			\emph{``DREAM5 -- Network Inference Challenge''}\\
			%\textsc{Team:} Visconti A., Esposito R., and Cordero F. (University of Turin)\\
			\textsc{Rank:}  The ensemble approach~\cite{Vis11b,Mar12} developed with the other team members (Roberto Esposito and Francesca Cordero) and which combines multiples approaches within a Naive Bayes classifier, despite its simplicity, ranked third for the reverse engineering of real organisms' gene regulatory networks and first for the reconstruction of the \emph{Saccharomyces Cerevisiae}'s network (out of 29 participants). 
		\end{minipage}}
\end{doubletablelist}




% --------------------------------
% TEACHING
% --------------------------------

\vspace{0.2cm}

\mediumtitle{Teaching activity}
\begin{doubletablelist}
	
	\dtlist{Dec 2019/present}{\textbf{Instructor} for several programming and data analysis workshops:\\
							& \hskip1cm - The Unix Shell\\
							& \hskip1cm - Version Control and collaboration with Git/Github\\
							& \hskip1cm - Python Programming\\
							& \hskip1cm - R Programming\\
							& \hskip1cm - Introduction to Working with Data\\
							& \hskip1cm - OpenRefine\\
							& These workshops are designed for PhD students and early career researchers but are open to researchers at every level, including PIs, are offered regularly (roughly twice a year), and have always been evaluated as ``excellent'' or ``exceptional'' by the attendees (details on some of the workshops can be found at \url{https://kcl-carpentries.github.io/}). The workshops are centred around the idea of ``live coding'', where attendees code along with the instructors thus getting useful hands-on experience and improving their ability to explore the topics on their own. }				
	
	\dtlist{Nov 2016/present}{\textbf{Co-organiser} of the Regulatory Genomics journal club at the Department of Twins Research \& Genetic Epidemiology, King's College London. The journal club discusses papers on latest achievements and methods in the fields of regulation of gene expression, epigenetics, splicing, evolution, and related topics. }
	
	\dtlist{a.y. 2021/2022}{ \textbf{Instructor} for a \emph{Version Control and collaboration with Git/Github} workshop at the UK Health Security Agency (UKHSA)\\
							&\textbf{Instructor} for \emph{The Unix Shell} and the \emph{Version Control and collaboration with Git/Github} workshops for students and researchers of the MRC Gambia and the AKU Nairobi Research Centres. Both workshops were rated as ``exceptional'' by the attendees.  \\
							&\textbf{Instructor} for the \emph{Metagenomics Data Analysis: Investigating the invisible world of microbes} (details at \url{https://alesssia.github.io/metagenomic_workshop/}). The workshop, which included frontal lesson and hands-on sessions, was designed for researchers at every level (including but not limited to PhD students, post-doctoral researchers and PIs) without any previous knowledge of the topics and tools presented.}

	\dtlist{a.y. 2013/2014}{\textbf{Teaching assistant} for the \emph{``Human Molecular Genetics''} MSc Department of Genomics of Common Diseases, Imperial College London. Alessia Visconti was offering support during the practical sessions on R programming as well as one-to-one meetings with students attending the following courses:\\
							%& \hskip0.5cm Workshops on: \\
							& \hskip1cm - The Unix Shell\\
							& \hskip1cm - R Programming\\
							& \hskip1cm - Exploratory Data Analysis and Probability\\
							& \hskip1cm - Quantitative genetics\\
							& \hskip1cm - Next Generation Sequencing Data Analysis.}				
\end{doubletablelist}							

% \newpage

{\footnotesize \noindent (Italian only)}

\begin{doubletablelist}
	\dtlist{a.y. 2013/2014}{\textbf{Lecturer} for the \emph{``Data analysis''} course, Department of Biological Science, University of Turin. Alessia Visconti was offering support during the practical sessions on R programming as well as one-to-one meetings with students. \\
							&\textbf{Lecturer} for the \emph{``Operating System''} course, Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions on R programming as well as one-to-one meetings with students.}
	\dtlist{a.y. 2012/2013}{\textbf{Lecturer} for the \emph{``Operating System and Networking''} course, Interfaculty School of Strategic Studies, University of Turin. Alessia Visconti was the sole responsible for the practical sessions covering the basis of GNU/Linux, the Unix shell, and process management. She designed the final project, \emph{i.e.}, the development of a basic client/server application in C (details at \url{https://alesssia.github.io/sistemi_elab_info_I_2012_13})\\
							&\textbf{Lecturer} for the \emph{``Operating System''} course,  Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions as well as one-to-one meetings with students. }
	\dtlist{a.y. 2011/2012}{\textbf{Lecturer} for the \emph{``Database''} course, Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions as well as one-to-one meetings with students.}
	\dtlist{a.y. 2010/2011}{\textbf{Lecturer} for the \emph{``Database''} course, Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions as well as one-to-one meetings with students. \\
							&\textbf{Lecturer} for the \emph{``Formal Language''} course, Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions as well as one-to-one meetings with students. \\
							&\textbf{Lecturer} for the \emph{``Statistics and data mining with SAS''} course, Department of Mathematics, University of Turin. Alessia Visconti prepared slides and recorded 3 hours of video lessons on SAS Enterprise Miner. She also prepared a self-evaluation questionnaire for the students.}
	\dtlist{a.y. 2009/2010}{\textbf{Lecturer} for the \emph{``Computer Science''} course, Department of Letters and Philosophy, University of Turin. Alessia Visconti was the sole responsible for the practical sessions covering the MS Office suite and the students' evaluation (details at \url{https://alesssia.github.io/lab_lettere_2009_10/}).}
	\dtlist{a.y. 2006/2007}{\textbf{Teaching assistant} for the \emph{``Program Languages - JAVA''} course, Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions. }
	\dtlist{a.y. 2005/2006}{\textbf{Teaching assistant} for the \emph{``Program Languages - JAVA''} course, Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions,}
	\dtlist{a.y. 2004/2005}{\textbf{Teaching assistant}  for the \emph{``Program Languages - C''} course, Department of Computer Science, University of Turin. Alessia Visconti was offering support during the practical sessions.}
\end{doubletablelist}

%\newpage

% --------------------------------
% METORING
% --------------------------------

\mediumtitle{Supervision activity}

% \noindent
% During her academic career, Alessia Visconti supervised: 4 BSc, 8 BSc, 2 post-master, and 5 PhD students as well as 2 postdocs. She also supervised 6 visiting research students (1 BSc, 2 MSc, 3 PhD).\\
%
% \noindent
% Thanks to her multidisciplinary experience, which combines a BSc, MSc, and PhD in computer science with more than 10 years of research activity in the biomedical field, Alessia Visconti has been able to supervise students with different backgrounds \emph{(e.g.}, computer science, bioinformatics, molecular biology, medicine, neurobiology, and engineering) offering a stimulating and engaging environment.

\begin{doubletablelist}	
	
	\dtlist{a.y. 2021/present}{\textbf{Co-supervisor} of Mr Roger Compte Boixader in his PhD project at King's College London. Roger's project aims at dissecting the aetiology of disk degeneration using a multi\emph{-omics} machine learning approach.}
	\dtlist{a.y. 2020/present}{\textbf{Assistance with the supervision} of Ms Karla Lee in her PhD project at King's College London. Karla's PhD project aims at investigating multi\emph{-omics} signatures of survival and/or response that could be used to identify patients with advanced melanoma who would benefit from treatment with immune checkpoint inhibitors (as described in~\cite{Ros22,Vis23}).}
	\dtlist{a.y. 2019/present}{\textbf{Assistance with the supervision} of Ms Xinyuan Zhang in her PhD project at King's College London. Xinyuan's PhD project aims at studying the interplay between the gut metagenome, medications, and diseases using multiple bioinformatics approaches (as described in~\cite{Zha22}).}
	\dtlist{a.y. 2016/2023}{\textbf{Co-supervisor} of Mr Niccol\`o Rossi during his research visit at King's College London. Niccol\`o's PhD project aimed at identifying and characterising the causes of lipid-metabolism disruption in patients with severe and unexplained familial dyslipidemia. The supervision continued during Niccol\`o's postdoc, which aimed at studying cardiovascular diseases and their risk factors with multiple bioinformatics approaches (as described in~\cite{Ros22}).}
	\dtlist{a.y. 2021/2022}{\textbf{Co-supervisor} of Ms Darvina Magandran in her MSc project in \emph{Microbiome in Health and Disease} at King's College London. Darvina's project aims at studying the interaction between the gut microbiome and eczema using machine learning approaches.}
	\dtlist{a.y. 2020/2021}{\textbf{Co-supervisor} of Ms Raphaela Joos in her MSc project in \emph{Microbiome in Health and Disease} at King's College London. Raphaela's project aims at studying the interaction between the presence of \emph{Lactobacillacee} in the faeces and blood lipid levels using machine learning approaches.\\
							&\textbf{Co-supervisor} of Ms Petra Blackburn in her MSc project in \emph{Microbiome in Health and Disease} at King's College London. Petra's project aims at investigating the interaction between hosts' gene expression and their gut microbiome.\\
							&\textbf{Co-supervisor} of Ms Natalie Falshaw in her MSc project in \emph{Microbiome in Health and Disease} at King's College London. Natalie's project aims at investigating the interaction between the hosts' DNA methylome and their gut microbiome.}
	\dtlist{a.y. 2019/2020}{\textbf{Co-supervision} of Mr Simon Couvreur in his PhD rotation project at King's College London. Simon's project aimed at studying the human glycome by developing new approaches for detecting co-abundant modules.\\
		&\textbf{Co-supervisor} of Ms Xinyu Huang in her MSc project in Pharmacology at King's College London. Xinyu's project aimed at studying the interaction between medications and the gut microbiome.\\
		&\textbf{Co-supervisor} of Ms Helen King in her PhD rotation project at King's College London. Helen's project aimed at dissecting the interaction between the gut microbiome and lipid levels in blood.}
	\dtlist{a.y. 2018/2020}{\textbf{Supervisor} of Ms Giulia Piaggeschi during her research visit at King's College London. Giulia's project, which was part of her PhD, aimed at studying immune cell modifications in peripheral blood and their interaction with tobacco smoking (as described in~\cite{Pia21}).}
	\dtlist{a.y. 2014/2020}{\textbf{Assistance with the supervision} of Ms Marianna Sanna in her PhD project at King's College London and her research activity at Imperial College London. Marianna's work aimed at dissecting the aetiology of melanoma and melanoma risk phenotypes using multiple bioinformatics approaches (as described in~\cite{Rib16,Duf17,Vis19a,Vis20,San20}).\\
	\dtlist{a.y. 2018/2019}{\textbf{Supervisor} of Mr Yuhao Lin's summer project as part of the King's Undergraduate Research Fellowships (KURF). Yuhao's project aimed at studying the human gut microbiome in healthy and diseased twins.}
			& \textbf{Supervisor} of Ms Olivia Castellini P\'erez's summer internship as part of the Erasmus+ program. Olivia's project aimed at studying epigenetic plasticity triggered by tobacco smoking.\\
			& \textbf{Co-supervisor} of Ms Miriam Margari's master thesis in Genomic Medicine at Imperial College London, entitled: ``Identification of novel genomic imprinting effects on gene expression in human tissues''.}
	\dtlist{a.y. 2017/2018}{\textbf{Supervisor} of Ms Lechun Huo's summer project as part of the King's Undergraduate Research Fellowships (KURF). Lechun's project aimed at identifying new statistical and computational approaches for studying the human gut microbiome in twins.\\
		& \textbf{Co-supervisor} of Dr Robin Mesnage's master thesis in Bioinformatics at Birkbeck University of London, entitled: ``A Metagenome-wide association study suggests that glycome composition associates with pathogenic bacteria abundance in the gut microbiome''.}
	\dtlist{a.y. 2016/2018}{\textbf{Assistance with the supervision} of Dr Harriet Cullen during her research fellowship at King's College London. Harriet's project aimed at dissecting the genetic and epigenetic basis of neuropsychiatric diseases following pre-term delivery using bioinformatics approaches (as described in~\cite{Cul18})}
	\dtlist{a.y. 2016/2017}{\textbf{Supervisor} of Ms Yuri Nemoto's summer project as part of the King's Undergraduate Research Fellowships (KURF). Yuri's project aimed at identifying connections between the human gut microbiome and lipid profiles in twins.\\
		& \textbf{Co-supervisor} of Ms Fudi Wang's research visit. Fudi's project aimed at identifying connections between facial ageing features and a set of genomic loci using novel bioinformatics approaches..}
	\dtlist{a.y. 2015/2016}{\textbf{Co-supervisor} of Ms Esther Kok's summer internship at the CERN OpenLab. Esther's project aimed at developing an efficient workflow to detect structural variants in DNA sequence data.}
	\dtlist{a.y. 2015/2019}{\textbf{Assistance with the supervision} of Mr Taghi Aliyev during his PhD at the CERN OpenLab. Taghi's project aimed at using the ROOT library for storing genomic data.}
	\dtlist{a.y. 2013/2014}{\textbf{Co-supervisor} of Mr Marcin \'Swistak's internship at Imperial College London. Marcin's project aimed at dissecting the genetic basis of melanoma and its connection with ageing using bioinformatics approaches.\\
		& \textbf{Co-supervisor} of Mr George Powell's master thesis in Human Molecular Genetics entitled: \emph{``Enrichment of Genomic Runs of Homozygosity for Copy Number Variation in Population Cohorts and Family Trios''}.}
	\dtlist{a.y. 2010/2011}{\textbf{Co-supervisor} of Mr Marco Gallizio's bachelor thesis in Computer Science at the University of Turin entitled: \emph{``A web interface for querying the Restructured Gene Ontology''}.}
\end{doubletablelist}	

 % \newpage

% --------------------------------
% REVIEWING
% --------------------------------

\mediumtitle{Review activity}
\begin{doubletablelist}
\dtlist{Editorial board membership}{ Since September 2019, Alessia Visconti is Review Editor of Human and Medical Genomics (speciality section of Frontiers in Genetics).}
\dtlist{Reviewer}{ Alessia Visconti serves as a reviewer for the following international journals: Bioinformatics, BioData Mining, BMC Cancer, BMC Nutrition, BMC Supplements, European Journal of Human Genetics, GigaScience, Journal of the European Academy of Dermatology and Venereology, Knowledge and Information Systems (KAIS), - Machine Learning, PLoS Computational Biology, PLoS One
        % & Alessia Visconti has served as a reviewer for the following international conferences/workshops:\\
% 	    & \hskip0.2cm - ACM International Conference on Information and Knowledge Management (CIKM)\\
% 		& \hskip0.2cm - Conference of the Italian Association for Artificial Intelligence (AI*IA)\\
% 		& \hskip0.2cm - Euromicro International Conference on Parallel, Distributed, and Network-Based Processing (PDP)\\
% 		& \hskip0.2cm - European Conference on Data Mining (ECDM)\\
% 		& \hskip0.2cm - European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases (ECML-PKDD)\\
% 		& \hskip0.2cm - IEEE International Conference on Data Mining (ICDM)\\
% 		& \hskip0.2cm - International Conference on Advanced Data Mining and Applications (ADMA)\\
% 		& \hskip0.2cm - International Conference on Artificial Immune Systems (ICARIS)\\
% 		& \hskip0.2cm - International Conference on Data Warehousing and Knowledge Discovery (DaWaK)\\
% 		& \hskip0.2cm - International Joint Conference on Knowledge Discovery, Knowledge Engineering and Knowledge Management (KDIR)\\
% 		& \hskip1cm - Network Tools and Applications in Biology (NETTAB)
}
\end{doubletablelist}
%

% --------------------------------
% MISCELLANEA
% --------------------------------

\mediumtitle{Outreach activity}


\begin{doubletablelist}
	\dtlist{Jul 2017 -- present}{Member and mentor of the \emph{Artificial Intelligence Club for Gender Minorities}, which aims at promoting gender diversity in the artificial intelligence and scientific community via meetups, and mentorship. Alessia Visconti organised workshops on collaborative data science via Git and GitHub. Since May 2018, she is also co-organising the group monthly journal club}
	\dtlist{Mar 2016 -- Sep 2017}{Member, tutor, and mentor of the \emph{RLadies London} community and of \emph{Researc[her] Research} community. These groups aim at promoting gender diversity in the R and STEM community via meetups, mentorship and global collaboration. With \emph{Researc[her]}, Alessia Visconti gave speeches on reproducibility, and workflow development}
\end{doubletablelist}


% --------------------------------
% ADMIN
% --------------------------------

% \newpage

\mediumtitle{University Responsibilities}

\begin{doubletablelist}

	\dtlist{Dec 2019 -- present}{Co-organiser of a series of workshops aiming at teaching basic computer literacy and data analysis skills with the KCL Carpentries, King's College London}
	\dtlist{Nov 2016 -- present}{Co-organiser of of the Regulatory Genomics journal club at the Department of Twins Research \& Genetic Epidemiology, King's College London.}
	\dtlist{Jan 2012 -- Dec 2013}{Faculty member as representative of postdoctoral fellows at the Department of Computer Science, University of Turin}
	\dtlist{Jan 2009 -- Dec 2011}{Faculty member as representative of PhD students at the Department of Computer Science, University of Turin}
	
\end{doubletablelist}


% % \newpage
% \mediumtitle{Other activities}
%
% \begin{doubletablelist}

% --------------------------------
% AWARDS

% \dtlist{Awards}{
% 	\vspace{-0.6cm}
% 	\begin{itemize} %[itemsep=-0.5ex]
%
% 		\minusitem \begin{minipage}{0.65\textwidth}
% 		 \textsc{Two-year Postdoctoral fellowship} at the Department of Computer Science, University of Turin. Founded by Italian Minister of Education, University and Research (MIUR) -- 2012-2013
% 		\end{minipage}
%
% 		\minusitem \begin{minipage}{0.65\textwidth}
% 		 \textsc{Postdoctoral training grant} (\euro3000). Founded by Regione Piemonte -- 2012
% 		\end{minipage}
%
% 		\minusitem \begin{minipage}{0.65\textwidth}
% 		 \textsc{Three-years PhD fellowship (best project)} at the School of Science and High Technology, Department of Computer Science, University of Turin. Founded by Italian Minister of Education, University and Research (MIUR) -- 2009-2011
% 		\end{minipage}
% 	\end{itemize}
% }

% --------------------------------
% CHALLENGES

% \dtlist{Participation in international challenges}{
% 	\vspace{-1.0cm}
% 	\begin{itemize} %[itemsep=-0.5ex]
% 		\graybulletitem   \begin{minipage}[t]{0.65\textwidth}
% 			\emph{``BioDataHack 2018 -- Genomic, Biodata and Improving Health Outcomes''}\\
% 			%\textsc{Team:} Visconti A., Jun Aruga, Oliver Giles, Ioannis Valasakis, Chen Zhang\\
% 			\textsc{Rank:} The project presented by Alessia Visconti and the other members of her team (Jun Aruga, Oliver Giles, Ioannis Valasakis e Chen Zhang) advanced the vision of a device that will allow the constant monitoring of IBDs by patients from the comfort of their own homes, ranked first on the ARM, Cavium, and Atos Challenge: \emph{How can we use mobile technology to transform biological data processing?}. During the two-day BioData Hackathon, the team successfully ported the metagenomics pipeline developed by Alessia Visconti~\cite{Vis18b} onto Arm’s 64-bit architecture, where the data could be processed in a few hours, showing that the analysis of microbial data can be successfully taken out of centralised data centres. The solution also implemented a neural network that, receiving as input the microbial profile produced by the analysis pipeline, could predict the disease status. Even with a very small set of training data, the proposed approach was able to separate patients with Crohn's disease and ulcerative colitis (the two main forms of IBD) from healthy controls with over 90\% accuracy. The team also proposed a derivative score from the neural network’s output. Such a score, which could be tracked over time, would allow individuals to measure the effects of lifestyle/medication changes on their disease’s progression to assess their efficacy in almost real time. The next step would be to use the data generated in this monitoring process to craft suggestive models, able to offer individuals advice and treatments based on what has been effective in patients with similar microbiome profiles.
% 			%1st on the ARM, Cavium, and Atos Challenge: \emph{How can we use mobile technology to transform biological data processing?}
% 		\end{minipage}
%
% 		\graybulletitem  \begin{minipage}[t]{0.65\textwidth}
% 			\emph{``DREAM6 -- Promoter Activity Prediction Challenge''}\\
% 			%\textsc{Team:} Visconti A. (University of Turin) and Workman C.T. (Technical University of Denmark)\\
% 			\textsc{Rank:} The proposed approach (developed with Ali Altıntas and Chris Workman) which combined results from two well-known machine learning approaches (regression trees and support vector machines for regression), ranked 8th out of 21 participants.
% 			%8th out of 21 participants
% 		\end{minipage}
%
% 		\graybulletitem   \begin{minipage}[t]{0.65\textwidth}
% 			\emph{``DREAM5 -- Network Inference Challenge''}\\
% 			%\textsc{Team:} Visconti A., Esposito R., and Cordero F. (University of Turin)\\
% 			\textsc{Rank:} The ensemble approach~\cite{Vis11b,Mar12} developed with the other team members (Roberto Esposito and Francesca Cordero) and which combines multiples approaches within a Naive Bayes classifier, despite its simplicity, ranked third for the reverse engineering of real organisms' gene regulatory networks and first for the reconstruction of the \emph{Saccharomyces Cerevisiae}'s network (out of 29 participants).
% 			% 15th out of 29 participants when both synthetic and real networks are considered, 3rd on real networks, and 1st on \emph{S. Cerevisiae}'s network
% 		\end{minipage}
% 	\end{itemize}
% }
	
% --------------------------------
% RESEARCH VISIT

% \dtlist{Research Visits}{
% 	\vspace{-0.6cm}
% 	\begin{itemize} %[itemsep=-0.5ex]
% 		\minusitem \begin{minipage}{0.65\textwidth}
% 		 \textsc{Jan 2015} -- visiting researcher at Weill Cornell Medical College in Qatar
% 		\end{minipage}
% 		\minusitem \begin{minipage}{0.65\textwidth}
% 		 \textsc{Jun - Dec 2011} -- visiting PhD student at the Regulatory Genomics Group, Center of Biological Sequence Analysis, Systems Biology Department, Technical University of Denmark, under the supervision of Prof. C. Workman
% 		\end{minipage}
% 	\end{itemize}
% }


% % --------------------------------
% % EXTRA
%
% \dtlist{Outreach activity}{
% 	\vspace{-0.7cm}
% 	\begin{itemize}[itemsep=-0.5ex]
% 		\graybulletitem \emph{Jul 2017 -- present : } member and mentor of the \emph{Artificial Intelligence Club for Gender Minorities}, which aims at promoting gender diversity in the artificial intelligence and scientific community via meetups, and mentorship. Alessia Visconti organised workshops on collaborative data science via Git and GitHub. Since May 2018, she is also co-organising the group monthly journal club.
% 		\graybulletitem  \emph{Mar 2016 -- Sep 2017 : } member, tutor, and mentor of the \emph{RLadies London} community and of \emph{Researc[her] Research} community. These groups aim at promoting gender diversity in the R and STEM community via meetups, mentorship and global collaboration. With \emph{Researc[her]}, Alessia Visconti gave speeches on reproducibility, and workflow development.
% 	\end{itemize}
% }
%
% \end{doubletablelist}
%



\newpage

% --------------------------------
% REASEARCH
% --------------------------------

\mediumtitle{Synopsis of Research}

\begin{itemize}
	
\bulletitem \smalltitle{\emph{-omics} of human diseases}\\
Advances in \emph{-omics} data collection created an opportunity to identify factors influencing the risk of common diseases. Alessia Visconti is mainly involved in a set of projects aiming at dissecting the aetiology of melanoma, melanoma risk phenotypes, and their connection with ageing~\cite{Rib16,Pui16,Hys18,Vis18a,Duf17,Vis19a,Vis20,Lan20,San20,Swi15}, and in predicting melanoma response to immunotherapy~\cite{Ros22,Vis23}.
Additionally, she uses systems biology and bioinformatics approaches to study IgA Nephropathy~\cite{Lom16,Dot21}, cognition and neurodevelopmental disease~\cite{Joh15,Cul18}, reading and language disabilities~\cite{Gia16}, epigenetic plasticity~\cite{Car16} and modification~\cite{Zag18}, thyroid diseases~\cite{Mar20}, atopic dermatitis~\cite{Gro21,Bud22}, cardiovascular diseases and their risk factors~\cite{Ros21}, immune system modifications~\cite{Pia21}, and X-inactivation~\cite{Zit23}. 
She also developed a framework for pairwise association testing in related samples~\cite{Vis16}, that has been used to perform one of the first epigenome-wide association studies in an Arab population~\cite{AlM15}. 
Recently, she has been investigating the influence of the gut microbiome on human health~\cite{Vis19,Bar20,LeR22,Zha22,Val23,Lou23}. She also reported on how to conduct metagenomic studies in microbiology and clinical research~\cite{Vis18c} and developed a novel pipeline which ensures reproducibility in metagenomics research~\cite{Vis18b}.

\bulletitem \smalltitle{SARS-CoV-2 research}\\
The SARS-CoV-2 virus is responsible for an acute respiratory illness.
Alessia Visconti was part of the team that performed the first data cleaning and analyses for the information collected by the COVID Symptom Tracker app (developed in collaboration with ZOE Ltd)~\cite{Mur21}. Several publications arose from this work~\cite{Men20,Lee20, Zaz20,Hop21,Wil21,Sud21}, and, in particular, she led two studies investigating skin manifestations of SARS-CoV-2~\cite{Vis21, Vis22}.

\bulletitem \smalltitle{Reverse engineering of gene regulatory networks}\\
The reverse engineering problem, \emph{i.e.}, the inference of gene regulatory networks from data, is a cardinal task on the biological research agenda.
Alessia Visconti worked on two approaches for the reverse engineering of gene regulatory networks and applied them to several model organisms. The first uses a Naive Bayes-based framework merging multiple pieces of information derived from microarray experiments~\cite{Mar12, Vis11b}. The second aims at deciphering temporal influences between genes and proteins~\cite{Vis12b}. 

\bulletitem \smalltitle{Data mining techniques for biological data analysis}\\
Data mining allows the extraction of previously unknown knowledge from large data sets.
Alessia Visconti developed novel techniques that allow the exploitation of domain knowledge and multiple data sources to improve co-clustering and bi-clustering results.
These have been used: \emph{i)}~to identify protein sequences characterised by common patterns~\cite{Vis08, Cor09a, Cor08b}, \emph{ii)}~to study synthenies in microorganisms~\cite{Bon11}, \emph{iii)}~to analyse RNA secondary structure~\cite{Cor08a}, and \emph{iv)}~to discover groups of genes showing similar expression profiles under the same set of experimental conditions~\cite{Vis13a, Cor09b, Vis11c, Vis12b}.

\bulletitem \smalltitle{Machine learning techniques for solving biological problems}\\
Machine learning focuses on the development of algorithms that improve through experience. An important application of Machine Learning  is the prediction of new knowledge from patterns learnt from data. 
Alessia Visconti leveraged and combined machine learning approaches to deal with several biological problems, such as: \emph{i)}~the prediction of promoter activities from promoter sequences, \emph{ii)}~the identification of pharmacogenes~\cite{Vis12a, Vis12b}, and \emph{iii)}~the study of peptide-based drugs~\cite{Erm14, Vis15a, Erm13a}.

\bulletitem \smalltitle{Gene Ontology restructuration} \\
The Gene Ontology (GO) represents a collaborative effort to provide a structured vocabulary for consistent gene descriptions. Although GO facilitates information retrieval, its structure may hide some useful knowledge, such as gene cooperation.
Alessia Visconti worked on a restructuration of Gene Ontology (RGO) that enhances automated analysis, such as gene profiling and clustering, statistical enrichment, as well as the evaluation of gene functional similarities~\cite{Vis11a, Vis10a, Vis12b}.

\bulletitem \smalltitle{Probabilistic Graphical Models}\\
Probabilistic Graphical Models (PGMs) sport a rigorous theoretical foundation and provide an abstract language for modelling application domains. Answering Maximum a Posteriori queries over a PGM entails finding the assignment to the graph variables that \emph{globally} maximises the probability of an observation. 
Alessia Visconti contributed to the development of a novel exact algorithm for answering Maximum a Posteriori queries on tree-structured PGMs~\cite{Esp13}.

\end{itemize}


\newpage

% --------------------------------
% PUBLICATIONS
% --------------------------------

\mediumtitle{Publications}

% \vspace{0.2cm}
%
% \noindent
% Alessia Visconti is co-author of 53 peer-reviewed scientific papers, of which 47 are published in international journals, 3 in LNAI, 2 in proceedings of international conferences, and 1 as a book chapter. She is the first and the senior author in 16 and 4, respectively. \\
%
% \noindent
% Alessia Visconti has 3,215 total citations, and an h-index of 19 (Source: Scopus, March 14th, 2023).
%
% \vspace{0.4cm}

{\small 
\noindent \emph{~$^{\textbf{$\dag $}}$ indicates that the authors contributed equally to the work}

\noindent \emph{~$^{\textbf{$\ddag $}}$ means that the authors jointly supervised the work}
}

\vspace{0.4cm}

\smalltitle{International Journals}

\indentp{
	\begin{itemize}
		
		\bibitem[J48]{Lou23} Louca P., Meijnikman A., Nogal MacHo A., Asnicar F., Attaye I., Vijay A., Kouraki A., \textbf{Visconti A.}, \dots, Bulsiewicz W., Nieuwdorp M., Valdes A., Menni C., \emph{The secondary bile acid isoursodeoxycholate is associated with postprandial lipaemia inflammation and appetite and changes post bariatric surgery}, Cell Reports Medicine, 2023, doi:10.1016/j.xcrm.2023.100993
		
		\bibitem[J47]{Zit23}  Zito A., Roberts A.L.$^{\textbf{$\dag $}}$, \textbf{Visconti A.}$^{\textbf{$\dag $}}$, Rossi N., Andres-Ejarque R., Nardone S., El-Sayed Moustafa J.S., Falchi M., and Small K.S., \emph{Escape from X-inactivation in twins exhibits intra- and inter-individual variability across tissues and is heritable}, PLoS Genetics, 2023, doi:10.1371/journal.pgen.1010556
				
		\bibitem[J46]{Vis23} \textbf{Visconti A.}, Rossi N., Deri\^s H., Lee K.A., ..., Sasieni P., Bataille V.$^{\textbf{$\ddag $}}$,  Lauc G.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{Total serum N‐glycans associate with response to immune checkpoint inhibition therapy and survival in patients with advanced melanoma}, BMC Cancer, 2023, doi:10.1186/s12885-023-10511-3
		
	  \bibitem[J45]{Val23} Valles-Colomer M., Blanco-Míguez A., Manghi P., Asnicar F., \dots, \textbf{Visconti A.}, \dots, Spector T.D., Domenici E., Collado M.C., and Segata N., \emph{The person-to-person transmission landscape of the gut and oral microbiomes}, Nature, 2023, doi:10.1038/s41586-022-05620-1
		
	  \bibitem[J44]{Zha22} Zhang X., Adebayo A.S., Wang D., Raza Y., Tomlinson M., Dooley H., Bowyer R.C.E., Small K., Steves C.J., Spector T.D., Duncan E.L., \textbf{Visconti A.}$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{PPI-induced changes in plasma metabolite levels influence total hip bone mineral density in a UK cohort}, Journal of Bone and Mineral Research, 2022, doi:10.1002/jbmr.4754
		
	  \bibitem[J43]{Ros22}  Rossi N.$^{\textbf{$\dag $}}$, Lee K. A.$^{\textbf{$\dag $}}$, Bermudez M.V., \textbf{Visconti A.}, Thomas A.M., Bolte L.A., \dots, Weersma R.K., Hospers G.A.P., Fehrmann R.S.N, Bataille V.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{Circulating inflammatory proteins associate with response to immune checkpoint inhibition therapy in patients with advanced melanoma}, EBioMedicine, 2022, doi:10.1016/j.ebiom.2022.104235
				
	   \bibitem[J42]{Vis22}	\textbf{Visconti A.}$^{\textbf{$\dag $}}$,, Murray B.$^{\textbf{$\dag $}}$, Rossi N., Wolf J., Ourselin S., Spector T.D., Freeman E.E., Bataille V.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$ \emph{Cutaneous Manifestations of SARS-CoV-2 infection during the Delta and Omicron waves in 348,691 UK users of the UK ZOE COVID Study App}, British Journal of Dermatology, 2022, doi:10.1111/bjd.21784
	   
	  \bibitem[J41]{LeR22} Le Roy C.I., Kurilshikov A., Leeming E.R., \textbf{Visconti A.}, Bowyer R.C.E, Menni C., Falchi M., Koutnikova H., Veiga P., Zhernakova A., Derrien M., and Spector T.D. \emph{Yoghurt consumption is associated with changes in the composition of the human gut microbiome and metabolome}, BMC Microbiology, 2022, doi:10.1186/s12866-021-02364-2
	
	  \bibitem[J40]{Mur21} Murray B., Kerfoot E., Chen L., Deng J., Graham M.S., Sudre C.H., Molteni E., Canas L.S., Antonelli M., Klaser K., \textbf{Visconti A.}, Hammers A., Chan A.T., Franks P.W., Davies R., Wolf J., Spector T.D., Steves C.J., Modat M., and Ourselin S. \emph{Accessible data curation and analytics for international-scale citizen science datasets}, Scientific Data, 2021, doi:10.1038/s41597-021-01071-x

	   \bibitem[J39]{Gro21} Grosche S.$^{\textbf{$\dag $}}$, Marenholz I.$^{\textbf{$\dag $}}$, Esparza-Gordillo J.$^{\textbf{$\dag $}}$, Arnau-Sole A.$^{\textbf{$\dag $}}$, \dots, \textbf{Visconti A.}, \dots, Worth CL, Xu CJ, and Lee YA, \emph{Rare variant analysis in eczema identifies exonic variants in DUSP1, NOTCH4 and SLC9A4}, Nature Communications, 2021, doi:10.1038/s41467-021-26783-x

	   \bibitem[J38]{Dot21} Dotz V.$^{\textbf{$\dag $}}$, \textbf{Visconti A.}$^{\textbf{$\dag $}}$, Lomax-Browne H.$^{\textbf{$\dag $}}$, Florent C.$^{\textbf{$\dag $}}$, Ederveen A.H., Medjeral-Thomas N., Cook H.T., Pickering M.,  Wuhrer M.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{O- and N-Glycosylation of Serum Immunoglobulin A is Associated with IgA Nephropathy and Glomerular Function}, Journal of the American Society of Nephrology, 2021, doi:10.1681/ASN.2020081208

		\bibitem[J37]{Sud21} Sudre C.H.$^{\textbf{$\dag $}}$, Lee K.A.$^{\textbf{$\dag $}}$ Lochlainn M.N.$^{\textbf{$\dag $}}$, Varsavsky T, Murray B., \dots, \textbf{Visconti A.}, \dots, Spector T.D., Steves C.J.$^{\textbf{$\ddag $}}$, and Ourselin S.$^{\textbf{$\ddag $}}$, \emph{Symptom clusters in COVID-19: A potential clinical prediction tool from the COVID Symptom Study app}, Science Advances, 2021, doi:10.1126/sciadv.abd4177
		
	\end{itemize}
}

\indentp{
	\begin{itemize}
		
		 \bibitem[J36]{Pia21} Piaggeschi G., Rolla S., Rossi N., Brusa D., Naccarati A., Couvreur S., Spector T.D., Roederer M., Mangino M., Cordero F., Falchi M.$^{\textbf{$\ddag $}}$ and \textbf{Visconti A.}$^{\textbf{$\ddag $}}$, \emph{Immune trait shifts in association with tobacco smoking: a study in healthy women}, Frontiers in immunology, 2021, doi:10.3389/fimmu.2021.637974
		
	   \bibitem[J35]{Ros21} Rossi N.$^{\textbf{$\dag $}}$, Aliyev E.$^{\textbf{$\dag $}}$, \textbf{Visconti A.}, Akil A.S.A., Syed N., Aamer W., Padmajeya S.S., Falchi M.$^{\textbf{$\ddag $}}$, and Fakhro K.A.$^{\textbf{$\ddag $}}$, \emph{Ethnic-specific association of amylase gene copy number with adiposity traits in a large Middle Eastern biobank}, Genomic Medicine, 2021, doi:10.1038/s41525-021-00170-3
		
 	   \bibitem[J34]{Wil21}	Williams F.M.K., Freidin M.B., Mangino M., Couvreur S., \textbf{Visconti A.}, Bowyer R.C.E., Le Roy C.I., Falchi M., Mompe\'o O., Sudre C., Davies R., Hammond C., Menni C., Steves C.J., and Spector T.D., \emph{Self-Reported Symptoms of COVID-19, Including Symptoms Most Predictive of SARS-CoV-2 Infection, Are Heritable}, Twin Research and Human Genetics, 2021, doi:10.1017/thg.2020.85
		
	    \bibitem[J33]{Vis21} \textbf{Visconti A.}$^{\textbf{$\dag $}}$, Bataille V.$^{\textbf{$\dag $}}$, Rossi N., Kluk J., Murphy R., Puig S., Nambi R., Bowyer R.C.E., Murray B., Bournot A., Wolf J., Ourselin S., Steves C., Spector T.D.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{Diagnostic value of cutaneous manifestation of SARS-CoV-2 infection}, British Journal of Dermatology, 2021, doi:10.1111/bjd.19807
		
		\bibitem[J32]{Hop21} Hopkinson N.S.$^{\textbf{$\dag $}}$, Rossi N.$^{\textbf{$\dag $}}$, El-Sayed Moustafa J., Laverty A.A., Quint J.K., Freidin M., \textbf{Visconti A.}, Murray B., Modat M., Ourselin S., Small K., Davies R., Wolf J., Spector T.D., Steves C.J.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{Current smoking and COVID-19 risk: results from a population symptom app in over 2.4 million people}, Thorax, 2021, doi:10.1136/thoraxjnl-2020-216422
		
		\bibitem[J31]{Bar20} Bar N.$^{\textbf{$\dag $}}$, Korem T.$^{\textbf{$\dag $}}$, Weissbrod O., Zeevi D., Rothschild D., Leviatan S., Kosower N., Lotan-Pompan M., Weinberger A., Le Roy C.I., Menni C., \textbf{Visconti A.}, Falchi M., Spector T.D., The IMI DIRECT consortium, Adamski J., Franks P.W., Pedersen O. and Segal E., \emph{A reference map of potential determinants for the human serum metabolome}, Nature, 2020, doi:10.1038/s41586-020-2896-2
		
		\bibitem[J30]{Zaz20} Zazzara M.B.$^{\textbf{$\dag $}}$, Penfold R.S.$^{\textbf{$\dag $}}$, Roberts A.L.$^{\textbf{$\dag $}}$, Lee, K.A., Dooley H., Sudre C.H., Welch C., Bowyer R.C.E, \textbf{Visconti A}, \dots, Martin F.C., Steves C.J.$^{\textbf{$\ddag $}}$, Lochlainn M.N.$^{\textbf{$\ddag $}}$, \emph{Probable delirium is a presenting symptom of COVID-19 in frail, older adults: a cohort study of 322 hospitalised and 535 community-based older adults}, Age and Ageing, 2020, doi:10.1093/ageing/afaa223
		
		\bibitem[J29]{San20} Sanna M.$^{\textbf{$\dag $}}$, Li X.$^{\textbf{$\dag $}}$, \textbf{Visconti A.}, Freidin M. B., Sacco C., Ribero S., Hysi P., Bataille V., Han J.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{Looking for Sunshine: Genetic Predisposition to Sun-Seeking in 265,000 Individuals of European Ancestry}, Journal of Investigative Dermatology, 2020, doi:10.1016/j.jid.2020.08.014

		\bibitem[J28]{Lee20} Lee K.A.$^{\textbf{$\dag $}}$, Ma W.$^{\textbf{$\dag $}}$, Sikavi D.R., \dots, \textbf{Visconti A.}, \dots, Ourselin S., Spector T.D., and Chan A.T., COPE consortium, \emph{Cancer and Risk of COVID-19 Through a General Community Survey}, Oncologist, 2020, doi:10.1634/theoncologist.2020-0572

		\bibitem[J27]{Sca20} Scarfi F., Orozco A.P., \textbf{Visconti A.}, and Bataille V., \emph{An Aggressive Clinical Presentation of Familial Leiomyomatosis Associated with a Fumarate Hydratase Gene Variant of Uncertain Clinical Significance}, Acta Dermato-venereologica, 2020, doi:10.2340/00015555-3573

		\bibitem[J26]{Men20} Menni C.$^{\textbf{$\dag $}}$, Valdes A.M.$^{\textbf{$\dag $}}$, Freidin M.B., Sudre C.H., Nguyen L.H., Drew, D.A., Ganesh S., Varsavsky T., Cardoso M.J., El-Sayed Moustafa J.S., \textbf{Visconti A.}, Hysi P., Bowyer R.C.E., Mangino M., Falchi M., Wolf J., Ourselin S., Chan A.T., Steves C.J.$^{\textbf{$\ddag $}}$, and Spector T.D.$^{\textbf{$\ddag $}}$, \emph{Real-time tracking of self-reported symptoms to predict potential COVID-19}, Nature Medicine, 2020, doi:10.1038/s41591-020-0916-2

		\bibitem[J25]{Lan20} Landi M.T., Bishop D.T., MacGregor S., \dots, \textbf{Visconti A.}, \dots, Shi J., Iles M.M. and Law M.H., \emph{Genome-wide association meta-analyses combining multiple risk phenotypes provide insights into the genetic architecture of cutaneous melanoma susceptibility}, Nature Genetics, 2020, doi:10.1038/s41588-020-0611-8

		\bibitem[J24]{Vis20} \textbf{Visconti A.}, Sanna M., Bataille V., and Mario F., \emph{Genetics plays a role in nevi distribution in women}, Melanoma Management, 2020, doi:10.2217/mmt-2019-0019 [Invited editorial]

		\bibitem[J23]{Mar20} Martin T.C., Illieva K.M., \textbf{Visconti A.}, Beaumont M., Kiddle S.J., Dobson R.J.B., Mangino M., Lim E.M., Pezer M., Steves C.J., Bell J.T., Wilson S.G., Lauc G., Roederer M., Walsh J.P., Spector T.D.$^{\textbf{$\ddag $}}$, Karagiannis S.N.$^{\textbf{$\ddag $}}$, \emph{Dysregulated Antibody, Natural Killer Cell and Immune Mediator Profiles in Autoimmune Thyroid Diseases}, MDPI Cells, 2020, doi:10.3390/cells9030665
		
	\end{itemize}
}


\indentp{
	\begin{itemize}

		\bibitem[J22]{Vis19} \textbf{Visconti A.}$^{\textbf{$\dag $}}$, Le Roy C.I.$^{\textbf{$\dag $}}$, Rosa F., Rossi N., Martin T.C., Mohney R.P., Li W., de Rinaldis E., Bell J.T., Venter J.C., Nelson K.E., Spector T.D.$^{\textbf{$\ddag $}}$, and Falchi M.$^{\textbf{$\ddag $}}$, \emph{Interplay between the human gut microbiome and host metabolism}, Nature Communications, 2019, doi:10.1038/s41467-019-12476-z

		\bibitem[J21]{Vis19a} \textbf{Visconti A.}, Ribero S., Sanna M., Spector T.D., Bataille V., and Mario F., \emph{Body site-specific genetic effects influence naevus count distribution in women}, Pigment Cell \& Melanoma Research, 2019, doi:10.1111/pcmr.12820
		
		\bibitem[J20]{Cul18} Cullen H., Krishnan M.L., Selzam S., Ball G., \textbf{Visconti A.}, Saxena A., Counsell S.J., Hajnal J., Breen G., Plomin R., and Edwards, A.D. \emph{Polygenic risk for neuropsychiatric disease and vulnerability to abnormal deep grey matter development}, Scientific Reports, 2019, doi:10.1038/s41598-019-38957-1
		
		\bibitem[J19]{Duf17} Duffy D., Zhu G., Li X., \dots, \textbf{Visconti, A.}, \dots, Falchi M., Han J.$^{\textbf{$\ddag $}}$, Martin N.G.$^{\textbf{$\ddag $}}$, Melanoma GWAS Consortium \emph{Novel pleiotropic risk loci for melanoma and nevus density implicate multiple biological pathways}, Nature Communications, 2018, doi:10.1038/s41467-018-06649-5
		
		\bibitem[J18]{Vis18c} Martin T.C.$^{\textbf{$\dag $}}$, \textbf{Visconti A}$^{\textbf{$\dag $}}$, Spector T.D., and Falchi M., \emph{Conducting metagenomic studies in microbiology and clinical research}, Applied Microbiology and Biotechnology, 2018, doi:10.1007/s00253-018-9209-9

		\bibitem[J17]{Vis18b} \textbf{Visconti A}, Martin T.C., and Falchi M., \emph{YAMP: a containerised workflow enabling reproducibility in metagenomics research}, GigaScience, 2018, doi:10.1093/gigascience/giy072
		
		\bibitem[J16]{Vis18a} \textbf{Visconti A.}, Duffy D., Liu F., Zhu G., \dots, Han J., Bataille V., and Falchi M., \emph{Genome-wide association study in 176,678 Europeans reveals genetic loci for tanning response to sun exposure}, Nature Communications, 2018, doi:10.1038/s41467-018-04086-y
		
		\bibitem[J15]{Hys18} Hysi P.G.$^{\textbf{$\dag $}}$, Valdes A.M.$^{\textbf{$\dag $}}$, Liu F.$^{\textbf{$\dag $}}$, Furlotte N.A., Evans D.M., Bataille V., \textbf{Visconti A.}, \dots, Kayser M.$^{\textbf{$\ddag $}}$, and Spector T.D.$^{\textbf{$\ddag $}}$, \emph{Genome-wide association meta-analysis of individuals of European ancestry identifies new loci explaining a substantial fraction of hair color variation and heritability}, Nature Genetics, 2018, doi:10.1038/s41588-018-0100-5
		
		\bibitem[J14]{Zag18} Zaghlool S.B., Mook-Kanamori D.O., Kader S., Stephan N., Halama A., Engelke R., Sarwath H., Al-Dous E.K., Mohamoud Y.A., Roemisch-Margl W., Adamski J., Kastenmüller G., Friedrich N., \textbf{Visconti A.}, \dots, Malek J.A., and Suhre K., \emph{Deep molecular phenotypes link complex disorders and physiological insult to CpG methylation}, Human Molecular Genetics, 2018, doi:10.1093/hmg/ddy006
				
		\bibitem[J13]{Vis16} \textbf{Visconti A.}, Al-Shafai M.,  Al Muftah W.A.,  Zaghlool S.B., Mangino M., Suhre K., and Falchi M., \emph{PopPAnTe: population and pedigree association testing for quantitative data}, BMC Genomics, doi:10.1186/s12864-017-3527-7
				
		\bibitem[J12]{Pui16} Puig-Butille J.A., Gimenez-Xavier P., \textbf{Visconti A.}, Nsengimana J., Garcia-Garcia F., Tell-Marti G., Escamez M.J., Newton-Bishop J.A., Bataille V., Del Rio M., Dopazo J., Falchi M, and Puig S., \emph{Genomic expression differences between cutaneous cells from red hair colour individuals and black hair colour individuals based on bioinformatic analysis.}, Oncotarget, 2016, doi:10.18632/oncotarget.14140
		
		\bibitem[J11]{Rib16} Ribero S.$^{\textbf{$\dag $}}$, Sanna M.$^{\textbf{$\dag $}}$, \textbf{Visconti A.}, Navarini A., Aviv A., Glass D., Spector T.D., Smith C., Simpson M., Barker J., Mangino M., Falchi M.$^{\textbf{$\ddag $}}$, and Bataille V.$^{\textbf{$\ddag $}}$, \emph{Acne and telomere length. A new spectrum between senescence and apoptosis pathways}, Journal of Investigative Dermatology, 2016, doi:10.1016/j.jid.2016.09.014

		\bibitem[J10]{Lom16} Lomax-Browne H.J.$^{\textbf{$\dag $}}$, \textbf{Visconti A.}$^{\textbf{$\dag $}}$, Pusey C.D., Cook H.T., Spector T.D., Pickering M.C$^{\textbf{$\ddag $}}$, and Falchi M$^{\textbf{$\ddag $}}$, \emph{IgA Glycosylation is Heritable in Healthy Twins}, Journal of the American Society of Nephrology, 2016, doi:10.1681/ASN.2016020184

		\bibitem[J9]{Gia16} Gialluisi A., \textbf{Visconti A.}, Willcutt E.G., Smith S.D., Pennington B.F. Falchi M., DeFries J.C.,  Olson R.K., Francks C., and Fisher S.E., \emph{Investigating the effects of copy number variants on reading and language performance}, Journal of Neurodevelopmental Disorders, 2016, doi:10.1186/s11689-016-9147-8

		\bibitem[J8]{AlM15} Al Muftah W.A.$^{\textbf{$\dag $}}$, Al-Shafai M.$^{\textbf{$\dag $}}$, Zaghlool S.B., \textbf{Visconti A.}, Tsai P.C., Kumar P., Spector T., Bell J.T., Falchi M.$^{\textbf{$\ddag $}}$, and Suhre K.$^{\textbf{$\ddag $}}$, \emph{Epigenetic associations of type 2 diabetes and BMI in an Arab population}, Clinical Epigenetics, 2016, doi:10.1186/s13148-016-0177-6

 	\end{itemize}
}


\indentp{
	\begin{itemize}
		
		\bibitem[J7]{Joh15} Johnson M.R., Shkura K., Langley S.R., \dots, \textbf{Visconti A.}, \dots, Kaminski R.M., Deary I.J., and Petretto E., \emph{Systems genetics identifies a convergent gene network for cognition and neurodevelopmental disease}, Nature Neuroscience, 2015, doi:10.1038/nn.4205

		\bibitem[J6]{Vis15a} \textbf{Visconti A.}, Ermondi G., Caron G., and Esposito R., \emph{Prediction and Interpretation of the Lipophilicity of Small Peptides}, Journal of Computer-Aided Molecular Design, 2015, doi:10.1007/s10822-015-9829-4
		
		\bibitem[J5]{Vis13a} \textbf{Visconti A.}, Cordero F., and Pensa R.G., \emph{Leveraging additional knowledge to support coherent bicluster discovery in gene expression data}, Intelligent Data Analysis, 2014, doi:10.3233/IDA-140671
	
		\bibitem[J4]{Erm14} Ermondi G., \textbf{Visconti A.}, Esposito R., and Caron G., \emph{The Block Relevance (BR) analysis supports the dominating effect of solutes hydrogen bond acidity on $\Delta \log P_{\text{oct-tol}}$}, European Journal of Pharmaceutical Sciences, 2014, doi:10.1016/j.ejps.2013.12.007
		
		\bibitem[J3]{Mar12} Marbach D., Costello J.C., K\"{u}ffner R., Vega N., Prill R.J., Camacho D., Allison K.R., \dots, \textbf{Visconti A.}, \dots, Kellis M., Collins J.J., and Stolovitzky G., \emph{Wisdom of crowds for robust gene network inference}, Nature Methods, 2012, doi:10.1038/nmeth.2016

		\bibitem[J2]{Vis11a} \textbf{Visconti A.}, Esposito R., and Cordero F., \emph{Restructuring the Gene Ontology to Emphasize Regulative Pathways and to Improve Gene Similarity Queries}, Int. J. Computational Biology and Drug Design, 2011, doi:10.1504/IJCBDD.2011.041411

		\bibitem[J1]{Bon11} Bonfante P., Cordero F., Ghignone S., Ienco D., Lanfranco L., Leonardi G., Meo R., Montani S., Roversi L., and \textbf{Visconti A.}, \emph{A Modular Database Architecture Enabled to Comparative Sequence Analysis}, LNCS Transactions on Large-Scale Data- and Knowledge-Centered Systems - TLDKS IV, LNCS 6990, 2011, doi:10.1007/978-3-642-23740-9\_6
 	\end{itemize}
}

\vspace{0.4cm}
% \newpage

\smalltitle{In proceeding}

\indentp{
	\begin{itemize}
		\bibitem[P5]{Esp13} Esposito R, Radicioni D.P., and \textbf{Visconti A.}, \emph{CDoT: optimizing MAP queries on trees}, In proceedings of AI*IA 2013: Advances in Artificial Intelligence, XIIIth Int. Conf. of the Italian Association for Artificial Intelligence, Turin, December 4-6, 2013, LNAI 8249, doi:10.1007/978-3-319-03524-6\_41

		\bibitem[P4]{Vis11b}  \textbf{Visconti A.}, Esposito R., and Cordero F., \emph{Tackling the DREAM Challenge for Gene Regulatory Networks Reverse Engineering}, In Proceedings of AI*IA 2011: Artificial Intelligence Around Man and Beyond, XIIth Int. Conf. of the Italian Association for Artificial Intelligence - Palermo, September 15-17, 2011, LNAI 6934, doi:10.1007/978-3-642-23954-0\_34

		\bibitem[P3]{Vis10a} \textbf{Visconti A.}, Cordero F., Botta M., Calogero R.A., \emph{Gene Ontology rewritten for computing gene functional similarity}, In Proceedings of the Fourth International Conferences on Complex, Intelligent and Software Intensive Systems, February 15-18, 2010, doi:10.1109/CISIS.2010.30

% 	\end{itemize}
% }
%
% \indentp{
% 	\begin{itemize}

		\bibitem[P2]{Cor09b} Cordero F., Pensa R.G, \textbf{Visconti A.}, Ienco D. and  Botta M., \emph{Ontology-driven Co-clustering of Gene Expression Data}, In proceedings of AI*IA 2009: Emergent Perspectives in Artificial Intelligence, XI Int. Conf. of the Italian Association for Artificial Intelligence - Reggio Emilia, December 9-12, 2009, LNAI 5883, doi:10.1007/978-3-642-10291-2\_43

		\bibitem[P1]{Cor09a} Cordero F., \textbf{Visconti A.}, and Botta M., \emph{A new protein motif extraction framework based on constrained co-clustering}, In Proceedings of the 24th Annual ACM Symposium on Applied Computing - March 8-12, 2009, doi:10.1145/1529282.1529445

	\end{itemize}
}

\vspace{0.4cm}
% \newpage

\smalltitle{Book Chapters}

\indentp{
	\begin{itemize}
		\bibitem[BC1]{Vis11c} \textbf{Visconti A.}, Cordero F., Ienco D., and Pensa R.G., \emph{Coclustering under Gene Ontology Derived Constraints for Pathway Identification}, Biological Knowledge Discovery Handbook: Preprocessing, Mining and Postprocessing of Biological Data, Mourad Elloumi and Albert Y. Zomaya (Eds.), 2014, doi:10.1002/9781118617151.CH27
	\end{itemize}
}

\vspace{0.4cm}
\newpage

\smalltitle{Pre Prints}

\indentp{
	\begin{itemize}
		
		\bibitem[PP2]{Bud22}  Budu-Aggrey A., Kilanowski A., Sobczyk M.K., Shringarpure S.S., Mitchell R., \dots,  \textbf{Visconti A.}, \dots, Holloway J.W., Min J., Brown S.J., Standl M., Paternoster L., \emph{European and multi-ancestry genome-wide association meta-analysis of atopic dermatitis highlights importance of systemic immune regulation}, medRxiv, 2022, doi:10.1101/2022.10.05.22279072
		
	\end{itemize}
}

% \newpage
\vspace{0.4cm}

\smalltitle{Selected Abstracts and Posters}

\indentp{
	\begin{itemize}
		\bibitem[A6]{Car16} Carnero-Montoro E., \textbf{Visconti A.}, Sacco C., Tsai P.C, Spector T.D, Falchi M., and Bell J.T., \emph{Environmentally-induced epigenetic variability is associated with metabolic traits}, American Society of Human Genetics, October 2016

		\bibitem[A5]{Swi15} \'Swistak M., \textbf{Visconti A.}, Falchi M., Bataille V., and Spector T.D., \emph{Differential expression and coexpression analysis across multiple tissues in twins}, EMBO Young Scientists Forum, Warsaw, July 2015	
		
		\bibitem[A4]{Erm13a} Ermondi G., Esposito R., \textbf{Visconti A.}, Visentin S., Vallaro M., Rinaldi L, and Caron G., \emph{Application of in-silico ``classical'' drug discovery tools to peptide research}, NovAliX Conference 2013, Biophysics in drug discovery, Strasbourg, October 2013
		
		\bibitem[A3]{Vis12a} \textbf{Visconti A.}, Calogero R.A, and Cordero F., \emph{Improving biomarker discovering for chemosensitivity prediction using an integrated approach}, 9th Annual Meeting of the Italian Society of Bioinformatics (BITS), EMBnet.journal, April 2012	
		
		\bibitem[A2]{Cor08b} Cordero F., \textbf{Visconti A.}, and Botta M., \emph{A web interface to extract protein motif by constrained co-clustering}, RECOMB Regulatory Genomics 2008, Boston, October 23-November 3, 2008
		
		\bibitem[A1]{Cor08a} Cordero F., \textbf{Visconti A.}, and Botta M., \emph{A motif extraction framework applied on RNA secondary structure}, Alternative Splicing Workshop - Milano, October 3, 2008
\end{itemize}
}

% \newpage
\vspace{0.4cm}

\smalltitle{Thesis}

\indentp{
\begin{itemize}
	\bibitem[T3]{Vis12b} \textbf{Visconti A.}, \emph{Systems Biology: Knowledge Discovery and Reverse Engineering}, PhD Thesis, Department of Computer Science, University of Turin, 2012
	
	\bibitem[T2]{Vis08} \textbf{Visconti A.}, \emph{SPOT: an algorithm for the extraction and the analysis of biological patterns}, Master Thesis, Department of Computer Science, University of Turin, 2008

	\bibitem[T1]{Vis06} \textbf{Visconti A.}, \emph{The Haskell language}, Bachelor Thesis, Department of Computer Science, University of Turin, 2006
	\end{itemize}
}

% \newpage
\vspace{0.5cm}


\begin{flushright}
London, \today
\end{flushright}

\end{document}

