\documentclass[a4paper,10pt]{article}

\usepackage[margin=0.8in]{geometry}
\usepackage[T1]{fontenc}
\usepackage[english]{babel}

\usepackage{color}
\definecolor{linkColor}{rgb}{0.2,0.4,0.6}

\usepackage[
	colorlinks,
	citecolor=linkColor,
	linkcolor=linkColor,
	menucolor=linkColor,
	urlcolor=linkColor,
]{hyperref}

\usepackage{amsfonts}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{array}
\usepackage{bold-extra}
\usepackage{enumitem}
\usepackage{longtable}
\usepackage{url}
\usepackage{xcolor}
\usepackage{eurosym}
\usepackage{lmodern} 

%%%% TITLE SETTING
\newcommand{\bigtitle}[1]{
	{\begin{center} 
		\huge \textsc{#1}\\[0.7ex]
		\hrule
	\end{center}}
}
\newcommand{\mediumtitle}[1]{
	\vspace{0.2cm}
	{\noindent
	\Large \textsc{#1}\\[-2ex]
	\hrule
	\vspace{0.2cm}}
}
\newcommand{\smalltitle}[1]{
	\vspace{0.1cm}
	{\noindent 
	\large \textsc{#1}}
	\vspace{0.1cm}
}


%%%% PARAGRAPH
\newcommand{\indentp}[1]{
	\vspace{0.2cm}
	\begin{minipage}{0.95\textwidth}
	{\noindent \hskip-1px #1}
	\end{minipage}
	\vspace{0.2cm}
}

%%%% LIST
%double item
\newcolumntype{A}{>{\raggedleft}p{0.22\textwidth}}
\newcolumntype{B}{p{0.7\textwidth}}
\newenvironment{doubletablelist}
{
	\vspace{-0.2cm}
	\begin{longtable}[!h]{AB}}{\end{longtable}
}
\newcommand{\dtlist}[2]{
\hspace{-3cm}
\noindent
	\begin{minipage}{0.22\textwidth}
	\begin{flushright}
	\textsc{#1}
	\end{flushright}
	\end{minipage}
	& #2\\[0.2cm]
}
%single item
\newcolumntype{a}{>{\raggedleft}p{0.14\textwidth}}
\newcolumntype{b}{p{0.8\textwidth}}
\newenvironment{singletablelist}
{	\vspace{-0.2cm}
	\begin{longtable}[!h]{ab}}{\end{longtable}
}
\newcommand{\stlist}[2]{
	\hspace{-3cm}
	\noindent
	\begin{minipage}{0.24\textwidth}
	\begin{flushright}
	\textsc{#1}
	\end{flushright}
	\end{minipage}
	& #2\\[0.2cm]
}

%%%% ITEM BULLET
\newcommand{\bulletitem}{\item[$\bullet$]}
\newcommand{\graybulletitem}{ \item[\textcolor{gray}{$\bullet$}]}
\newcommand{\circitem }{\item[$\circ$]}
\newcommand{\minusitem}{\item[-]}
\newcommand{\astitem}{\item[$\ast$]}
\newcommand{\noitem}{\item[]}

% --------------------------------

\begin{document}

\begin{center}
\bigtitle{Alessia Visconti, PhD}
\end{center}

\vspace{0.2cm}

\noindent
\textbf{Assistant professor (tenure track)}\\
Center for Biostatistics, Epidemiology, and Public Health\\
Department of Clinical and Biological Sciences, University of Turin\\
Regione Gonzole 10, 10043, Orbassano, Torino, Italy\\

\noindent
\textsc{e-mail:} \href{mailto:alessia.visconti@unito.it}{alessia.visconti@unito.it}\\
\textsc{web:} \url{https://alesssia.github.io}

\vspace{0.2cm}

\mediumtitle{Research Interests}
\begin{itemize}[itemsep=-0.5ex]
	\minusitem  \textsc{Biostatistics \& Bioinformatics}
 	\minusitem  \textsc{Computational Biology \& Medicine}
 	\minusitem  \textsc{Data Mining \& Machine Learning}
	\minusitem  \textsc{Multi\emph{-omics} data analysis and integration}
	% \minusitem  \textsc{High-Throughput Biology}
	% \minusitem  \textsc{Complex Disease Genetics}
	% \minusitem  \textsc{Big Data}
	\minusitem  \textsc{Research Software Engineering}
\end{itemize}

\mediumtitle{Brief description}
\indentp{
Alessia Visconti is an expert in medical statistics and computational medicine. Her research activity deals with the development and application of statistical and computational methods to identify multi\emph{-omics} modifications influencing complex human phenotypes. 
She has also worked on the problem of knowledge discovery in biological data, developing new approaches tailored to solve biological tasks, and on the reverse engineering of gene regulatory networks.
}

\vspace{0.4cm}

% --------------------------------
% EDUCATION
% --------------------------------

\mediumtitle{Education}
\begin{singletablelist}
	%\stlist{Jul 2012}{\textbf{PhD in Science and High Technology}, University of Turin.\\
	\stlist{Jul 2012}{\textbf{PhD in Computer Science}, University of Turin.\\
	&\textsc{Thesis title:} \emph{Systems Biology: Knowledge Discovery and Reverse Engineering} }
	% &\textsc{Advisors}: prof. Marco Botta, Dr. Roberto Esposito}
	
	\stlist{Jul 2008}{\textbf{Master degree in Computer Science} \emph{``summa cum laude''}, University of Turin.\\
	&\textsc{Thesis title:} \emph{SPOT: an algorithm for the extraction and the analysis of biological patterns} }
	% &\textsc{Advisor}: prof. Marco Botta}

	\stlist{Mar 2006}{\textbf{Bachelor degree in Computer Science}  \emph{``summa cum laude''}, University of Turin.\\
	&\textsc{Thesis title:} \emph{The Haskell language} }
	% &\textsc{Advisor}:  prof. Viviana Bono}
\end{singletablelist}

% --------------------------------
% SKILLS
% --------------------------------

\mediumtitle{Skills}
\begin{doubletablelist}
	\dtlist{Language Skills}{\textsc{Italian}: native proficiency \\ 
							&\textsc{English}: professional working proficiency\\
							&\textsc{Greek (modern)}: elementary proficiency}
	\dtlist{Computing Skills}{\textsc{Programming languages}: bash, C, C++, JAVA, php, python, R, ruby\\
							&\textsc{Other languages}: CSS, \LaTeX, HTML, PyQt, XML\\
							&\textsc{Statistical software}: R, SAS\\
							&\textsc{Database management}: MySQL, MariaDB\\
							&\textsc{Version control systems \& reproducibility }: GIT, nextflow, docker, singularity\\
							&\textsc{Bioinformatics \& Genetic analysis}: BBmap, BEDTools, DESeq2, GCTA, GWAMA, LDAK, limma, lmekin, metal, Merlin, PLINK, QTDT, samtools, vcftools, \dots\\
							&\textsc{Structural Equation Modelling}: openMX, Mplus\\
							&\textsc{Data visualisation}: dot, ggplot2}
\end{doubletablelist}

\newpage


\input{research_posts}


% \mediumtitle{Career breaks}
% \begin{doubletablelist}
%     \dtlist{Aug 2022 - Apr 2023}{\textbf{Maternity leave}
% 	}
% \end{doubletablelist}


% --------------------------------
% AWARDS
% --------------------------------

\input{awards}

% --------------------------------
% PROJECTS
% --------------------------------

\input{projects}

\newpage

\mediumtitle{Review activity}

\begin{singletablelist}

\stlist{Editorial board membership}{ Since September 2019, Alessia Visconti is \emph{Review Editor} of Human and Medical Genomics (speciality section of Frontiers in Genetics).\\ &}
\stlist{Reviewer}{ Alessia Visconti serves as a reviewer for the following international journals: Bioinformatics, BioData Mining, BMC Cancer, BMC Nutrition, BMC Supplements, British Journal of Cancer, Computational and Structural Biotechnology Journal, Communications Biology, European Journal of Human Genetics, Frontiers in Cellular and Infection Microbiology, GigaScience, Gut Microbes, Journal of the European Academy of Dermatology and Venereology, Knowledge and Information Systems (KAIS), Machine Learning, PLoS Computational Biology, PLoS One, Nature Communications, Scientific Reports
        % & Alessia Visconti has served as a reviewer for the following international conferences/workshops:\\
% 	    & \hskip0.2cm - ACM International Conference on Information and Knowledge Management (CIKM)\\
% 		& \hskip0.2cm - Conference of the Italian Association for Artificial Intelligence (AI*IA)\\
% 		& \hskip0.2cm - Euromicro International Conference on Parallel, Distributed, and Network-Based Processing (PDP)\\
% 		& \hskip0.2cm - European Conference on Data Mining (ECDM)\\
% 		& \hskip0.2cm - European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases (ECML-PKDD)\\
% 		& \hskip0.2cm - IEEE International Conference on Data Mining (ICDM)\\
% 		& \hskip0.2cm - International Conference on Advanced Data Mining and Applications (ADMA)\\
% 		& \hskip0.2cm - International Conference on Artificial Immune Systems (ICARIS)\\
% 		& \hskip0.2cm - International Conference on Data Warehousing and Knowledge Discovery (DaWaK)\\
% 		& \hskip0.2cm - International Joint Conference on Knowledge Discovery, Knowledge Engineering and Knowledge Management (KDIR)\\
% 		& \hskip1cm - Network Tools and Applications in Biology (NETTAB)
}
\end{singletablelist}



\mediumtitle{Invited talks}

\begin{singletablelist}
	\stlist{17 Jun 2019}{\emph{``Reproducible shotgun metagenomic analysis with Nextflow and containers''} at the 1st London Bioinformatics Frontiers Frontiers conference, London, UK }
	\stlist{23 Nov 2018}{\emph{``Nextflow on the go''} at the 2nd Nextflow workshop, Barcelona, Spain }
	\stlist{6 Dec 2017}{\emph{``YAMP: a framework enabling reproducibility in metagenomics research''} at the Computational Biology seminar series, Francis Crick Institute, London, UK }
	\stlist{13 Dec 2017}{\emph{``My reproducible adventure in (meta)genomics (A tale of workflow development, research reproducibility \& open science)''} at the Researc/hers code seminar series, London, UK}
	\stlist{15 Sep 2017}{\emph{``Simplifying shotgun metagenomics analysis with Nextflow''} at the 1st Nextflow workshop, Barcelona, Spain}
	\stlist{7 Jul 2012}{\emph{``Knowledge-driven Co-clustering of Gene Expression Data''} at the Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark}
	\stlist{8 May 2009}{\emph{``Using a priori knowledge for the reverse engineering of gene regulatory networks''} at the Computer Laboratory, Cambridge University, Cambridge, UK}
\end{singletablelist}


\mediumtitle{Development of research software}

\noindent The software is available at \url{https://github.com/alesssia} or upon request.

\vspace{0.2cm}

\begin{singletablelist}
	\vspace{-1cm}
	\stlist{AID-ISA}{extracts biologically relevant biclusters from microarray gene expression data by leveraging additional knowledge (described in~\cite{Vis13a})}
	\stlist{CDoT}{is a novel exact algorithm for answering Maximum a Posteriori queries on tree-structured Probabilistic Graphical Models (described in~\cite{Esp13} and developed in collaboration with Roberto Esposito)}
	\stlist{famCNV (v2.0)}{enables genome-wide association of copy number variants with quantitative phenotypes in families (developed in collaboration with Mario Falchi)}
	\stlist{GOClust}{performs a co-clustering of microarray gene expression data using Gene Ontology-derived constraints (described in~\cite{Cor09b})}
	\stlist{MotifsLinker}{associates clusters of proteins with their frequent motifs} % (described in~\cite{Cor08a})}
	\stlist{PicNic}{extracts topologies and discovers patterns in sets of chemical compound}
	\stlist{PopPAnTe}{enables pairwise association testing in related samples (described in~\cite{Vis16})}
	\stlist{RGO}{is a reorganization of the Gene Ontology emphasising  regulative information and providing better structure for gene functional analysis (described in~\cite{Vis11a})}
	\stlist{SPOT}{performs an exhaustive search of frequent motifs in sets of biological sequences (described in~\cite{Vis08})}
	\stlist{YAMP}{allows processing raw metagenomic sequencing data up to the functional annotation (described in~\cite{Vis18b})}
\end{singletablelist}

\vspace{0.2cm}

\newpage

\mediumtitle{Service activity}

\begin{doubletablelist}
	    \dtlist{Jan 2025 -- present}{\textbf{Co-coordinator} of the Working Group in Machine Learning in Clinical Research of the Italian Society of Medical Statistics and Clinical Epidemiology (SISMEC, \url{https://www.sismec.info/sismec/gruppi-di-lavoro/gruppi-di-lavoro-sismec/machine-learning-nella-ricerca-clinica/})}
		\dtlist{Oct 2024 -- present}{\textbf{Member} of the Scientific Board for the Medical Residency program in Medical Statistics and Biometrics, at the University of Turin}
		\dtlist{Jun 2024 -- present}{\textbf{Member} of the Scientific Board for the PhD program in Complex Systems for Quantitative Biomedicine, at the University of Turin (\url{phd-csqb.campusnet.unito.it})}
		\dtlist{Dec 2023 -- present}{\textbf{DISC} (Department Internal Staff Contact) for the platform MedCapu (\url{www.medcap.unito.it}) at the University of Turin}
		\dtlist{Dec 2019 -- Sep 2023}{\textbf{Co-organiser} of a series of workshops aiming at teaching basic computer literacy and data analysis skills with the KCL Carpentries, King's College London (\url{kcl-carpentries.github.io})}
		\dtlist{Nov 2016 -- Jul 2022}{\textbf{Co-organiser} of the Regulatory Genomics journal club at the Department of Twins Research \& Genetic Epidemiology, King's College London.}
		\dtlist{Jan 2012 -- Dec 2013}{\textbf{Faculty member} as representative of postdoctoral fellows at the Department of Computer Science, University of Turin}
		\dtlist{Jan 2009 -- Dec 2011}{\textbf{Faculty member} as representative of PhD students at the Department of Computer Science, University of Turin}
	
\end{doubletablelist}


\mediumtitle{Outreach activity}

\begin{doubletablelist}
	\dtlist{Jul 2017 -- Feb 2023}{Member and mentor of the \emph{Artificial Intelligence Club for Gender Minorities}, which aims at promoting gender diversity in the artificial intelligence and scientific community via meetups, and mentorship. Alessia Visconti organised workshops on collaborative data science via Git and GitHub. Since May 2018, she is also co-organising the group monthly journal club\\ &}
	\dtlist{Mar 2016 -- Sep 2017}{Member, tutor, and mentor of the \emph{RLadies London} community and of \emph{Researc[her] Research} community. These groups aim at promoting gender diversity in the R and STEM community via meetups, mentorship and global collaboration. With \emph{Researc[her]}, Alessia Visconti gave speeches on reproducibility, and workflow development}
\end{doubletablelist}


% --------------------------------
% TEACHING
% --------------------------------

\input{teaching}

% \newpage

% --------------------------------
% MENTORING
% --------------------------------

\input{supervision}

 % \newpage

	







% \newpage
%
% % --------------------------------
% % REASEARCH
% % --------------------------------
%
% \mediumtitle{Synopsis of Research}
%
% \begin{itemize}
%
% \bulletitem \smalltitle{\emph{-omics} of human diseases}\\
% Advances in \emph{-omics} data collection created an opportunity to identify factors influencing the risk of common diseases. Alessia Visconti is mainly involved in a set of projects aiming at dissecting the aetiology of melanoma, melanoma risk phenotypes, and their connection with ageing~\cite{Rib16,Pui16,Hys18,Vis18a,Duf17,Vis19a,Vis20,Lan20,San20,Swi15}, and in predicting melanoma response to immunotherapy~\cite{Ros22,Vis23}.
% Additionally, she uses systems biology and bioinformatics approaches to study IgA Nephropathy~\cite{Lom16,Dot21}, cognition and neurodevelopmental disease~\cite{Joh15,Cul18}, reading and language disabilities~\cite{Gia16}, epigenetic plasticity~\cite{Car16} and modification~\cite{Zag18}, thyroid diseases~\cite{Mar20}, atopic dermatitis~\cite{Gro21,Bud22}, cardiovascular diseases and their risk factors~\cite{Ros21}, immune system modifications~\cite{Pia21}, and X-inactivation~\cite{Zit23}.
% She also developed a framework for pairwise association testing in related samples~\cite{Vis16}, that has been used to perform one of the first epigenome-wide association studies in an Arab population~\cite{AlM15}.
% Recently, she has been investigating the influence of the gut microbiome on human health~\cite{Vis19,Bar20,LeR22,Zha22,Val23,Lou23,Nog23a,Nog23b}. She also reported on how to conduct metagenomic studies in microbiology and clinical research~\cite{Vis18c} and developed a novel pipeline which ensures reproducibility in metagenomics research~\cite{Vis18b}.
%
% \bulletitem \smalltitle{SARS-CoV-2 research}\\
% The SARS-CoV-2 virus is responsible for an acute respiratory illness.
% Alessia Visconti was part of the team that performed the first data cleaning and analyses for the information collected by the COVID Symptom Tracker app (developed in collaboration with ZOE Ltd)~\cite{Mur21}. Several publications arose from this work~\cite{Men20,Lee20, Zaz20,Hop21,Wil21,Sud21}, and, in particular, she led two studies investigating skin manifestations of SARS-CoV-2~\cite{Vis21, Vis22}.
%
% \bulletitem \smalltitle{Reverse engineering of gene regulatory networks}\\
% The reverse engineering problem, \emph{i.e.}, the inference of gene regulatory networks from data, is a cardinal task on the biological research agenda.
% Alessia Visconti worked on two approaches for the reverse engineering of gene regulatory networks and applied them to several model organisms. The first uses a Naive Bayes-based framework merging multiple pieces of information derived from microarray experiments~\cite{Mar12, Vis11b}. The second aims at deciphering temporal influences between genes and proteins~\cite{Vis12b}.
%
% \bulletitem \smalltitle{Data mining techniques for biological data analysis}\\
% Data mining allows the extraction of previously unknown knowledge from large data sets.
% Alessia Visconti developed novel techniques that allow the exploitation of domain knowledge and multiple data sources to improve co-clustering and bi-clustering results.
% These have been used: \emph{i)}~to identify protein sequences characterised by common patterns~\cite{Vis08, Cor09a, Cor08b}, \emph{ii)}~to study synthenies in microorganisms~\cite{Bon11}, \emph{iii)}~to analyse RNA secondary structure~\cite{Cor08a}, and \emph{iv)}~to discover groups of genes showing similar expression profiles under the same set of experimental conditions~\cite{Vis13a, Cor09b, Vis11c, Vis12b}.
%
% \bulletitem \smalltitle{Machine learning techniques for solving biological problems}\\
% Machine learning focuses on the development of algorithms that improve through experience. An important application of Machine Learning  is the prediction of new knowledge from patterns learnt from data.
% Alessia Visconti leveraged and combined machine learning approaches to deal with several biological problems, such as: \emph{i)}~the prediction of promoter activities from promoter sequences, \emph{ii)}~the identification of pharmacogenes~\cite{Vis12a, Vis12b}, and \emph{iii)}~the study of peptide-based drugs~\cite{Erm14, Vis15a, Erm13a}.
%
% \bulletitem \smalltitle{Gene Ontology restructuration} \\
% The Gene Ontology (GO) represents a collaborative effort to provide a structured vocabulary for consistent gene descriptions. Although GO facilitates information retrieval, its structure may hide some useful knowledge, such as gene cooperation.
% Alessia Visconti worked on a restructuration of Gene Ontology (RGO) that enhances automated analysis, such as gene profiling and clustering, statistical enrichment, as well as the evaluation of gene functional similarities~\cite{Vis11a, Vis10a, Vis12b}.
%
% \bulletitem \smalltitle{Probabilistic Graphical Models}\\
% Probabilistic Graphical Models (PGMs) sport a rigorous theoretical foundation and provide an abstract language for modelling application domains. Answering Maximum a Posteriori queries over a PGM entails finding the assignment to the graph variables that \emph{globally} maximises the probability of an observation.
% Alessia Visconti contributed to the development of a novel exact algorithm for answering Maximum a Posteriori queries on tree-structured PGMs~\cite{Esp13}.
%
% \end{itemize}
%
%
\newpage

% --------------------------------
% PUBLICATIONS
% --------------------------------

\input{publication}


% \newpage
\vspace{0.5cm}


\begin{flushright}
Turin, \today
\end{flushright}

\end{document}

