% --------------------------------
% PROJECTS
% --------------------------------

\mediumtitle{Research collaborations}

\vspace{0.2cm}

\noindent Only collaborators outside King's College London (for projects started between 2014 and 2023) and the University of Turin (for projects started before 2014 and after 2024) are reported.


\vspace{0.2cm}

\smalltitle{Projects for which Alessia Visconti has scientific responsibility}

\begin{singletablelist}
	
	\stlist{2025}{
		  \textsc{Title:} \emph{``Read Assessment and Decision Support for ICU Readmission Prediction (READ-ICU)"}\\
		& \textsc{Budget:} \euro{43,200} -- \textsc{Funder:} \emph{Horizon Enfield} \\
		& \textsc{Collaborators:} National Technical University, Athens, Greece \\
		& \textsc{Objectives:} Developing a deep-learning ensemble predictive model to evaluate the risk of ICU readmission using MIMIC-IV and eICU datasets.\\
		& \textsc{Role:} co-PI. Within this project, she is co-supervising a PhD student (Emanuele Koumantakis).\\
		&
	}
	
	% \end{singletablelist}
	%
	% \newpage
	%
	% \begin{singletablelist}
	
	\stlist{2024-2025}{
		  \textsc{Title:} \emph{``Exploring the potential role of gut microbiota in improving outcomes of chronic myeloid leukaemia patients"}\\
		& \textsc{Budget:} \euro{80,000} -- \textsc{Funder:} \emph{ESH John Goldman Research Award}  \\
		& \textsc{Collaborators:} Italian Institute for Genomic Medicine, Turin, Italy \\
		& \textsc{Objectives:} Understanding the cross-talk between gut microbiota and immune response and its role in treatment tolerance and success in patients with chronic myeloid leukaemia. \\
		& \textsc{Role:} Responsible of the work-packages dealing with all the bioinformatics analyses. \\
		& 
	}
	
		%     \stlist{2022-2025}{
		%   \textsc{Title:} \emph{``Challenging the Dogma of Homogeneity in Gestational Diabetes Mellitus''}\\
		% & \textsc{Funder:} \emph{MRC Medical Research Council} -- \textsc{Budget:} £1,090,268  \\
		% & \textsc{Role:} In this project, which is now in the phase of data collection and aims at characterising pathophysiologically distinct subtypes of gestational diabetes (GDM), Alessia Visconti will develop and use bioinformatics and machine learning approaches to \emph{(a)}~evaluate similarities and differences in GDM subtypes in women of White European and South Asian descent, identifying variables (clinical/biochemical) that distinguish between subtypes, and \emph{(b)}~explore the relationships between subtypes and maternal/fetal/neonatal outcomes.
		% }
	
	\stlist{2022-2025}{
		  \textsc{Title:} \emph{``A Collaborative Approach to the Borne Uterine Mapping Programme (BUMP) Feasibility Study''}\\
		& \textsc{Funder:} \emph{BORNE} -- \textsc{Budget:} £500,000 \\
		& \textsc{Collaborators:} Imperial College London, London, UK, Newcastle University, Newcastle upon Tyne, UK, and University of Turin, Turin, Italy \\
		& \textsc{Objectives:} Studying normal term labour and map the uterus using state-of-the-art transcriptomics (bulk and single cells/nuclei), proteomics (bulk and single-proteomics), and spatial sequencing. \\
		& \textsc{Role:} Responsible for the work-packages aiming at the development of \emph{(a)}~pipelines for the analysis of single-cell and single-nucleus RNA sequencing and spatial transcriptomic data, and \emph{(b)}~deep-learning approaches for modelling their interaction. \\
		& 
	} 
		
	\stlist{2021-2024}{ 
		  \textsc{Title:} \emph{``Understanding phenotype and mechanisms of spontaneous preterm birth in sub-Saharan Africa (PRECISE-SPTB)''}\\
		& \textsc{Funder:} \emph{MRC Medical Research Council} -- \textsc{Budget:} £458,80 \\
		& \textsc{Collaborators:} Aga Khan University, Nairobi, Kenya, MRC Unit The Gambia at LSHTM, Banjul, The Gambia, and Eduardo Mondlane University, Maputo, Mozambique \\
		& \textsc{Objectives:} Determining the epidemiological and contextual nature of spontaneous preterm birth in three sub–Saharan African countries (Kenya, Nairobi and Mozambique), while developing technical infrastructure and training research scientists. \\
		& \textsc{Role:} Responsible for the preparation and delivery of workshops covering both basic and specialistic skills, namely: shell programming, version control and collaboration with Git/GitHub, programming in R, programming in python, workflow development with Nextflow, machine-learning approaches for biomedical data analysis, and metagenomic data analysis. Workshops have been rated as “exceptional” by the attendees. \\
		&
		}
		
	\stlist{2020-2021}{ 
		  \textsc{Title:} \emph{``A multi-omics study to dissect the role of the gut microbiome in IgA nephropathy risk''}\\
		& \textsc{Funder:} \emph{King's College London - Peking University Health Science Centre Joint Institute for Medical Research} --- \textsc{Budget:} £74,000 \\
		& \textsc{Collaborators:} Peking University, Peking, China \\
		& \textsc{Objectives:} Identify microbes and microbial functions associated with IgAN, and investigating their role in the disease by studying their interplay with blood glycomics and faecal metabolomics. \\
		& \textsc{Role:} Responsible for the work package dealing with \emph{in silico} characterisation and validation, using bioinformatics models, of microbes associated with IgA nephropathy and/or IgA glycosylation profiles. \\
		& 
		}
		
	\stlist{2016-2018}{ 
		  \textsc{Title:} \emph{``A high-resolution map of copy number and structural variation in Qatari genomes and their contribution to quantitative traits and disease''}\\
		& \textsc{Funder:} \emph{Qatar Foundation} -- \textsc{Budget:} £160,521  \\
		& \textsc{Collaborators:} Sidra Medical and Research Centre, Doha, Qatar \\
		& \textsc{Objectives:} Generating a database of structural variants in Qataris and leveraging the deep phenotyping on these samples to estimate, among the others, the contribution of SVs to CVD and related cardio-metabolic traits. \\
		& \textsc{Role:} Responsible for work-packages aiming at  the development of \emph{(a)}~an approach for the storage of genomic data taking advantage of the ROOT library, and \emph{(b)}~an ensemble approach for the identification of structural variation. She also conducted the bioinformatics analysis for~\cite{Ros21,Ros24} and supervised a postdoctoral researcher (Nicco\`o Rossi).\\
		&
	}
		
\end{singletablelist}


\smalltitle{Projects to which Alessia Visconti participates as researcher}

\noindent  Alessia Visconti has a prominent role in all projects, as shown by the number of publications in which she is first/last author.

\vspace{0.1cm}

\begin{singletablelist}
	
	\stlist{2023-2027}{
		\textsc{Title:} \emph{``PUZZLE: aPproccio integrato alla mUltimorbidit\`a: ricerca di base, traslaZionale, clinica e formaZione muLtidisciplinarE''}\\
		% & \textsc{Funder:} \emph{Italian Minister of Education, University and Research (MIUR)}\\
		& \textsc{Objectives:} Developing a novel model for the study, management, and care of patients with multi-morbidity.\\
		& \textsc{Role:} Leading researcher in the application of Deep Learning approaches to identify multi-morbidity patterns from epidemiological data on several diseases (\emph{e.g.}, cardiometabolic diseases and cancer) and environmental exposures (\emph{e.g.}, smoking, alcohol consumption, physical activity). Within this project, she is supervising a master student (Triparna Day).\\
		&
	}

	\stlist{2022-2025}{
		\textsc{Title:} \emph{``TrustAlert''}\\
		% & \textsc{Funder:} \emph{Compagnia di San Paolo}\\
		& \textsc{Collaborators:} Bruno Kessler Foundation, Trento, Italy, Links Foundation, Turin, Italy, Innovo Group, Turin, Italy, Local Health Authority Alba-Bra, Alba, Italy, Cottolengo Hospital, Turin, Italy\\
		& \textsc{Objectives:} Developing AI-based solutions for \emph{(a)}~real-time identification of health emergencies, \emph{(b)}~mapping vulnerable and multi-morbid populations, and \emph{(c)}~crate a \emph{living lab} for micro and macro simulations.\\
		& \textsc{Role:} Co-leading researcher in the application of Deep Learning approaches for \emph{(a)}~the extraction reliable clinical information from structured and unstructured data for supporting medical decision~\cite{Con24}, and \emph{(b)}~the mapping of morbidity and medical vulnerabilities patterns that exist within communities~\cite{Con25}. Within this project, she is supervising a master student (Francesca Rondinone), and two research assistants (Emanuele Pietropaolo, Giuseppe Martino).\\
		&
	}
	
		
	\stlist{2022-2025}{
		\textsc{Title:} \emph{``Repertor.IO: Incorporating Patient Preference Studies into Clinical Research and Decision Model''}\\
		% & \textsc{Funder:} \emph{Compagnia di San Paolo}\\
		& \textsc{Collaborators:} University of Padua, Padua, Italy\\
		& \textsc{Objectives:} Developing a platform for the collection, annotation, and analysis of patients preference studies.\\
		& \textsc{Role:} Contributing to the data analysis~\cite{DiB24}.\\
		&
	}	
		

	\stlist{2021-2022}{
		\textsc{Title:} \emph{``Predicting Response to Immunotherapy for Melanoma with gut Microbiome and metabolomics - The PRIMM Study''}\\
		% & \textsc{Funder:} \emph{Seerave Foundation}\\
		& \textsc{Collaborators:} University Medical Center, Groningen, The Netherland and University of Trento, Trento, Italy\\
		& \textsc{Objectives:} Predicting which patients with melanoma would benefit from the use of immune checkpoint inhibitors using multi-omics data. \\
		& \textsc{Role:} Leading researcher for the bioinformatics analysis for the identification of glyco-markers~\cite{Vis23} (first author) and collaborated to the analysis of proteomic data~\cite{Ros22} aiming at finding novel biomarkers of response and survival to identify those patients with melanoma who are most likely to benefit from immune checkpoint inhibitors. Within this project, she helped with the supervision of a PhD student (Karla Lee). \\
		& 
	}

% \end{singletablelist}
%
% % \newpage
%
% \begin{singletablelist}
	
	\stlist{2019-2021}{ 
		\textsc{Title:} \emph{``Dissecting the mechanisms of immune-mediated inflammation: a systems-immunology approach''}\\
		% & \textsc{Funder:} \emph{MRC Medical Research Council}\\
		% & \textsc{Collaborators:} \\
		& \textsc{Objectives:} Identifying targetable networks of circulating immune cells and cytokines involved in the normal immune-mediated inflammation process, and identify those disrupted in immune-mediated inflammatory diseases. \\
		& \textsc{Role:} Leading researcher for the bioinformatics analyses aiming at \emph{(a)}~the reverse engineering of immune cell co-expression networks and their involvement in a set of autoimmune diseases, and \emph{(b)}~the identification of genetic variations and microbes/metabolites responsible for the development of such diseases. Within this project, she supervised a postdoctoral researcher (Nicco\`o Rossi).\\
		& 
	}
	
	\stlist{2018-2024}{ 
		\textsc{Title:} \emph{``TREatment of ATopic eczema (TREAT) Registry Taskforce''}\\
		% & \textsc{Funder:} \emph{Chronic Disease Research Foundation}\\
		& \textsc{Collaborators:} more than 30 centers over 13 countries (see the registry website for details \url{treat-registry-taskforce.org}) \\
		& \textsc{Objectives:} studying the genetic and environmental basis of atopic dermatitis (consortium). \\
		& \textsc{Role:} Leading bioinformatician for the TwinsUK cohort~\cite{Gro21,Bud22,Sta25}. \\
		& 
	}

	\stlist{2016-2018}{ 
		\textsc{Title:} \emph{``Gut microbiome modulation of fasting glucose homeostasis and postprandial glycaemic response in TwinsUK and PREDICT: towards personalised diet for healthy aging''}\\
		% & \textsc{Funder:} \emph{Chronic Disease Research Foundation}\\
		& \textsc{Collaborators:} University of Trento, Trento, Italy\\
		& \textsc{Objectives:} studying the influence of the gut microbiome on cardiometabolic health. \\
		& \textsc{Role:} Alessia Visconti \emph{(a)}~developed a tool for the analysis of metagenomic data which ensures the reproducibility of the scientific results~\cite{Vis18b} (first author), \emph{(b)} performed the bioinformatics analysis of metagenomics and metabolomics data~\cite{Vis19,Bar20} (co-first author in the first study), \emph{(c)}~collaborated to further studies~\cite{Lou23,Nog23a,Nog23b}, and \emph{(d)}~supervised a PhD student for the work described in~\cite{Zha22} (co-senior author). \\
		& 
	}

	\stlist{2014-2016}{  
		\textsc{Title:} \emph{``An integrative genomics approach for non-invasive diagnostic biomarkers discovery in IgA nephropathy''}\\
		% & \textsc{Funder:} \emph{MRC Medical Research Council}\\
		& \textsc{Collaborators:} Imperial College London, London, UK\\
		& \textsc{Objectives:} Identify glycomics and genetic biomarkers for helping IgAN diagnosis and treatment.\\
		& \textsc{Role:} Alessia Visconti applied statistical and bioinformatics approaches for studying the role of IgA and its glycosylation profiles in the development of IgA nephropathy, as described in~\cite{Lom16,Dot21,Vis24} (co-first author in all).\\
		&
	}
	
	\stlist{2013-2015}{ 
		\textsc{Title:} \emph{``Senescence and melanoma -- An integrative systems biology approach to characterise the link between reduced biological senescence and melanoma susceptibility''}\\
		% & \textsc{Funder:} \emph{British Skin Foundation}\\
		& \textsc{Collaborators:} Hospital Clinic of Barcelona, Barcelona, Spain, QIMR Berghofer Medical Research Institute, Brisbane, Australia, Beijing Institute of Genomics, Beijing, China, Institute of Cancer and Pathology, Leeds, UK, Erasmus MC University Medical Center, Rotterdam, The Netherland, Women's Hospital, Boston, US.\\
		& \textsc{Objectives:} Identifying the genetics basis of melanoma skin cancer and of its related phenotypes. \\
		& \textsc{Role:} Alessia Visconti applied statistical and bioinformatics approaches for studying melanoma, melanoma risk phenotypes, and their connection with ageing, as described in~\cite{Pui16,Rib16,Hys18,Vis18a,Duf17,Vis19a,Vis20,San20,Lan20} (first author in three manuscripts).\\
		&
	}
				
	\stlist{2013-2018}{ 
		\textsc{Title:} \emph{``Genomic analysis of Type 2 Diabetes in Qatar, towards diabetes personalized medicine''}\\
		% & \textsc{Funder:} \emph{Qatar Foundation}\\
		& \textsc{Collaborators:} Weill Cornell Medicine-Qatar, Doha, Qatar\\
		& \textsc{Objectives:} Identifying the genetics and epigenetics basis of Type 2 Diabetes \\
		& \textsc{Role:} Alessia Visconti \emph{(a)}~developed and implemented an approach for the population and pedigree association testing for quantitative data~\cite{Vis16} (first author), and \emph{(b)}~conducted the bioinformatics analysis for~\cite{AlM15} and~\cite{Zag18}. \\
		&
	}
	
	\stlist{2012-2013}{
		\textsc{Title:} \emph{``LIMPET -- Isotropic And Anisotropic Lipophilicity To Model Permeability Of New Therapeutic Peptides''}\\
		% & \textsc{Funder:} \emph{Compagnia di San Paolo}\\
		% & \textsc{Collaborators:} \\
		& \textsc{Objectives:} Applying machine Learning approaches to predicts peptides permeability on the basis of experimental non-conventional lipophilicity indexes. \\
		& \textsc{Role:} Alessia Visconti \emph{(a)}~evaluated the ability of some combinations of descriptors/algorithms to find the best model to predict the lipophilicity of small peptides~\cite{Vis15a} (first author), and \emph{(b)}~performed the bioinformatics analyses to measure the ability of more than 200 compounds of acting as hydrogen bond donors~\cite{Erm14}.\\
		&
	}	

	\stlist{2007-2011}{
		\textsc{Title:} \emph{``BioBITs -- Developing white and green biotechnologies by converging platforms from biology and information technology towards metagenomics''}\\
		% & \textsc{Funder:} \emph{Regione Piemonte}\\
		& \textsc{Collaborators:} University of Eastern Piedmont, Alessandria, Italy\\
		& \textsc{Objectives:} Identifying and characterising populations of uncultivable bacteria living inside a symbiotic, arbuscular mycorrhizal fungus.\\
		& \textsc{Role:} Alessia Visconti contributed to the development of a modular framework for the analysis of metagenomics sequences, and was responsible for the co-clustering module~\cite{Bon11}. \\
		&
	}
	
	% \dtlist{2004-2009}{
	% 	\textsc{Title:} \emph{``Realizzazione di modelli informatici per la valorizzazione della qualit\`a e la tracciabilit\`a delle produzioni in specie da frutto coltivate in Piemonte''}\\
	% 	% & \textsc{Funder:} \emph{Regione Piemonte}\\
	% 	% & \textsc{Collaborators:} \\
	% 	& \textsc{Objectives:}  \\
	% 	& \textsc{Role:} Alessia Visconti contributed to the development of computational approaches for the classification and traceability of fruits produced in Piemonte.\\
	% 	&
	% }
			
\end{singletablelist}

% \newpage

\smalltitle{Projects with industrial partners to which Alessia Visconti participates}

\begin{singletablelist}

	\stlist{2020 - 2022}{
		\textsc{Partner:} ZOE Ltd \url{https://health-study.joinzoe.com/})\\
		& \textsc{Role:}  Alessia Visconti \emph{(a)}~developed the analysis pipeline for the daily data provided by more than four million users~\cite{Mur21}, \emph{(b)}~led two studies investigating skin manifestations of SARS-CoV-2~\cite{Vis21, Vis22}, and \emph{(c)}~performed the bioinformatics analysis for several other studies~\cite{Men20,Lee20,Zaz20,Hop21,Wil21,Sud21}.\\
		&
	}	

	\stlist{2019 - 2020}{
		\textsc{Partner:} Sanofi (\url{https://www.sanofi.com}) (``Sanofi iAwards Europe 2019'') \\
		& \textsc{Role:} Alessia Visconti carried out part of the bioinformatics analyses, and supervised a postdoctoral researcher (Niccol\`o Rossi).\\
		&
	}

\end{singletablelist}

	
\begin{singletablelist}
		
	\stlist{2018 - 2020}{
	\textsc{Partner:} Danone Nutricia Research (\url{https://www.nutriciaresearch.com}) \\
	& \textsc{Role:} Alessia Visconti performed the metagenomic data analyses, as described in~\cite{LeR22}.\\
	&
	}

\end{singletablelist}